[[05/11/2025 09:13:00 AM]] 
new run with parameters:
{'attention_mixture_components': 10,
 'batch_size': 32,
 'batch_sizes': [32],
 'beta1_decay': 0.9,
 'beta1_decays': [0.9],
 'checkpoint_dir': 'checkpoints_style_synthesis',
 'early_stopping_steps': 2000,
 'enable_parameter_averaging': False,
 'grad_clip': 10,
 'keep_prob_scalar': 1.0,
 'learning_rate': 0.0001,
 'learning_rates': [0.0001],
 'log_dir': 'logs_style_synthesis',
 'log_interval': 50,
 'logging_level': 20,
 'loss_averaging_window': 100,
 'lstm_size': 400,
 'min_steps_to_checkpoint': 2000,
 'num_restarts': 0,
 'num_training_steps': 30000,
 'optimizer': 'rms',
 'output_mixture_components': 20,
 'output_units': 121,
 'patiences': [2000],
 'prediction_dir': 'predictions_style_synthesis',
 'reader': None,
 'regularization_constant': 0.0,
 'restart_idx': 0,
 'style_embedding_size': 256,
 'validation_batch_size': 32,
 'warm_start_init_step': 0}
[[05/11/2025 09:13:02 AM]] all parameters:
[[05/11/2025 09:13:02 AM]] [('Variable:0', []),
 ('Variable_1:0', []),
 ('Variable_2:0', []),
 ('style_encoder/bidirectional/lstm_cell/kernel:0', [3, 512]),
 ('style_encoder/bidirectional/lstm_cell/recurrent_kernel:0', [128, 512]),
 ('style_encoder/bidirectional/lstm_cell/bias:0', [512]),
 ('style_encoder/bidirectional/lstm_cell_1/kernel:0', [3, 512]),
 ('style_encoder/bidirectional/lstm_cell_1/recurrent_kernel:0', [128, 512]),
 ('style_encoder/bidirectional/lstm_cell_1/bias:0', [512]),
 ('style_encoder/dense/kernel:0', [256, 1]),
 ('style_encoder/dense/bias:0', [1]),
 ('style_encoder/dense_1/kernel:0', [256, 256]),
 ('style_encoder/dense_1/bias:0', [256]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell/kernel:0', [732, 1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell/bias:0', [1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/attention/weights:0', [732, 30]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/attention/biases:0', [30]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell_1/kernel:0',
  [1132, 1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell_1/bias:0', [1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell_2/kernel:0',
  [1132, 1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell_2/bias:0', [1600]),
 ('rnn_style/gmm/weights:0', [400, 121]),
 ('rnn_style/gmm/biases:0', [121]),
 ('style_encoder/bidirectional/lstm_cell/kernel/RMSProp:0', [3, 512]),
 ('style_encoder/bidirectional/lstm_cell/kernel/RMSProp_1:0', [3, 512]),
 ('style_encoder/bidirectional/lstm_cell/recurrent_kernel/RMSProp:0',
  [128, 512]),
 ('style_encoder/bidirectional/lstm_cell/recurrent_kernel/RMSProp_1:0',
  [128, 512]),
 ('style_encoder/bidirectional/lstm_cell/bias/RMSProp:0', [512]),
 ('style_encoder/bidirectional/lstm_cell/bias/RMSProp_1:0', [512]),
 ('style_encoder/bidirectional/lstm_cell_1/kernel/RMSProp:0', [3, 512]),
 ('style_encoder/bidirectional/lstm_cell_1/kernel/RMSProp_1:0', [3, 512]),
 ('style_encoder/bidirectional/lstm_cell_1/recurrent_kernel/RMSProp:0',
  [128, 512]),
 ('style_encoder/bidirectional/lstm_cell_1/recurrent_kernel/RMSProp_1:0',
  [128, 512]),
 ('style_encoder/bidirectional/lstm_cell_1/bias/RMSProp:0', [512]),
 ('style_encoder/bidirectional/lstm_cell_1/bias/RMSProp_1:0', [512]),
 ('style_encoder/dense/kernel/RMSProp:0', [256, 1]),
 ('style_encoder/dense/kernel/RMSProp_1:0', [256, 1]),
 ('style_encoder/dense/bias/RMSProp:0', [1]),
 ('style_encoder/dense/bias/RMSProp_1:0', [1]),
 ('style_encoder/dense_1/kernel/RMSProp:0', [256, 256]),
 ('style_encoder/dense_1/kernel/RMSProp_1:0', [256, 256]),
 ('style_encoder/dense_1/bias/RMSProp:0', [256]),
 ('style_encoder/dense_1/bias/RMSProp_1:0', [256]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell/kernel/RMSProp:0',
  [732, 1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell/kernel/RMSProp_1:0',
  [732, 1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell/bias/RMSProp:0', [1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell/bias/RMSProp_1:0',
  [1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/attention/weights/RMSProp:0',
  [732, 30]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/attention/weights/RMSProp_1:0',
  [732, 30]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/attention/biases/RMSProp:0', [30]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/attention/biases/RMSProp_1:0',
  [30]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell_1/kernel/RMSProp:0',
  [1132, 1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell_1/kernel/RMSProp_1:0',
  [1132, 1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell_1/bias/RMSProp:0',
  [1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell_1/bias/RMSProp_1:0',
  [1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell_2/kernel/RMSProp:0',
  [1132, 1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell_2/kernel/RMSProp_1:0',
  [1132, 1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell_2/bias/RMSProp:0',
  [1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell_2/bias/RMSProp_1:0',
  [1600]),
 ('rnn_style/gmm/weights/RMSProp:0', [400, 121]),
 ('rnn_style/gmm/weights/RMSProp_1:0', [400, 121]),
 ('rnn_style/gmm/biases/RMSProp:0', [121]),
 ('rnn_style/gmm/biases/RMSProp_1:0', [121])]
[[05/11/2025 09:13:02 AM]] trainable parameters:
[[05/11/2025 09:13:02 AM]] [('style_encoder/bidirectional/lstm_cell/kernel:0', [3, 512]),
 ('style_encoder/bidirectional/lstm_cell/recurrent_kernel:0', [128, 512]),
 ('style_encoder/bidirectional/lstm_cell/bias:0', [512]),
 ('style_encoder/bidirectional/lstm_cell_1/kernel:0', [3, 512]),
 ('style_encoder/bidirectional/lstm_cell_1/recurrent_kernel:0', [128, 512]),
 ('style_encoder/bidirectional/lstm_cell_1/bias:0', [512]),
 ('style_encoder/dense/kernel:0', [256, 1]),
 ('style_encoder/dense/bias:0', [1]),
 ('style_encoder/dense_1/kernel:0', [256, 256]),
 ('style_encoder/dense_1/bias:0', [256]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell/kernel:0', [732, 1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell/bias:0', [1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/attention/weights:0', [732, 30]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/attention/biases:0', [30]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell_1/kernel:0',
  [1132, 1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell_1/bias:0', [1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell_2/kernel:0',
  [1132, 1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell_2/bias:0', [1600]),
 ('rnn_style/gmm/weights:0', [400, 121]),
 ('rnn_style/gmm/biases:0', [121])]
[[05/11/2025 09:13:02 AM]] trainable parameter count:
[[05/11/2025 09:13:02 AM]] 5070128
[[05/11/2025 09:13:02 AM]] built graph
[[05/11/2025 09:13:02 AM]] Restoring parameters from final_checkpoints/model-10350
[[05/11/2025 09:13:02 AM]] Started server process [1249]
[[05/11/2025 09:13:02 AM]] Waiting for application startup.
[[05/11/2025 09:13:02 AM]] Application startup complete.
[[05/11/2025 09:13:05 AM]] 
new run with parameters:
{'attention_mixture_components': 10,
 'batch_size': 32,
 'batch_sizes': [32],
 'beta1_decay': 0.9,
 'beta1_decays': [0.9],
 'checkpoint_dir': 'checkpoints_style_synthesis',
 'early_stopping_steps': 2000,
 'enable_parameter_averaging': False,
 'grad_clip': 10,
 'keep_prob_scalar': 1.0,
 'learning_rate': 0.0001,
 'learning_rates': [0.0001],
 'log_dir': 'logs_style_synthesis',
 'log_interval': 50,
 'logging_level': 20,
 'loss_averaging_window': 100,
 'lstm_size': 400,
 'min_steps_to_checkpoint': 2000,
 'num_restarts': 0,
 'num_training_steps': 30000,
 'optimizer': 'rms',
 'output_mixture_components': 20,
 'output_units': 121,
 'patiences': [2000],
 'prediction_dir': 'predictions_style_synthesis',
 'reader': None,
 'regularization_constant': 0.0,
 'restart_idx': 0,
 'style_embedding_size': 256,
 'validation_batch_size': 32,
 'warm_start_init_step': 0}
[[05/11/2025 09:13:08 AM]] all parameters:
[[05/11/2025 09:13:08 AM]] [('Variable:0', []),
 ('Variable_1:0', []),
 ('Variable_2:0', []),
 ('style_encoder/bidirectional/lstm_cell/kernel:0', [3, 512]),
 ('style_encoder/bidirectional/lstm_cell/recurrent_kernel:0', [128, 512]),
 ('style_encoder/bidirectional/lstm_cell/bias:0', [512]),
 ('style_encoder/bidirectional/lstm_cell_1/kernel:0', [3, 512]),
 ('style_encoder/bidirectional/lstm_cell_1/recurrent_kernel:0', [128, 512]),
 ('style_encoder/bidirectional/lstm_cell_1/bias:0', [512]),
 ('style_encoder/dense/kernel:0', [256, 1]),
 ('style_encoder/dense/bias:0', [1]),
 ('style_encoder/dense_1/kernel:0', [256, 256]),
 ('style_encoder/dense_1/bias:0', [256]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell/kernel:0', [732, 1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell/bias:0', [1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/attention/weights:0', [732, 30]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/attention/biases:0', [30]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell_1/kernel:0',
  [1132, 1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell_1/bias:0', [1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell_2/kernel:0',
  [1132, 1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell_2/bias:0', [1600]),
 ('rnn_style/gmm/weights:0', [400, 121]),
 ('rnn_style/gmm/biases:0', [121]),
 ('style_encoder/bidirectional/lstm_cell/kernel/RMSProp:0', [3, 512]),
 ('style_encoder/bidirectional/lstm_cell/kernel/RMSProp_1:0', [3, 512]),
 ('style_encoder/bidirectional/lstm_cell/recurrent_kernel/RMSProp:0',
  [128, 512]),
 ('style_encoder/bidirectional/lstm_cell/recurrent_kernel/RMSProp_1:0',
  [128, 512]),
 ('style_encoder/bidirectional/lstm_cell/bias/RMSProp:0', [512]),
 ('style_encoder/bidirectional/lstm_cell/bias/RMSProp_1:0', [512]),
 ('style_encoder/bidirectional/lstm_cell_1/kernel/RMSProp:0', [3, 512]),
 ('style_encoder/bidirectional/lstm_cell_1/kernel/RMSProp_1:0', [3, 512]),
 ('style_encoder/bidirectional/lstm_cell_1/recurrent_kernel/RMSProp:0',
  [128, 512]),
 ('style_encoder/bidirectional/lstm_cell_1/recurrent_kernel/RMSProp_1:0',
  [128, 512]),
 ('style_encoder/bidirectional/lstm_cell_1/bias/RMSProp:0', [512]),
 ('style_encoder/bidirectional/lstm_cell_1/bias/RMSProp_1:0', [512]),
 ('style_encoder/dense/kernel/RMSProp:0', [256, 1]),
 ('style_encoder/dense/kernel/RMSProp_1:0', [256, 1]),
 ('style_encoder/dense/bias/RMSProp:0', [1]),
 ('style_encoder/dense/bias/RMSProp_1:0', [1]),
 ('style_encoder/dense_1/kernel/RMSProp:0', [256, 256]),
 ('style_encoder/dense_1/kernel/RMSProp_1:0', [256, 256]),
 ('style_encoder/dense_1/bias/RMSProp:0', [256]),
 ('style_encoder/dense_1/bias/RMSProp_1:0', [256]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell/kernel/RMSProp:0',
  [732, 1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell/kernel/RMSProp_1:0',
  [732, 1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell/bias/RMSProp:0', [1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell/bias/RMSProp_1:0',
  [1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/attention/weights/RMSProp:0',
  [732, 30]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/attention/weights/RMSProp_1:0',
  [732, 30]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/attention/biases/RMSProp:0', [30]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/attention/biases/RMSProp_1:0',
  [30]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell_1/kernel/RMSProp:0',
  [1132, 1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell_1/kernel/RMSProp_1:0',
  [1132, 1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell_1/bias/RMSProp:0',
  [1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell_1/bias/RMSProp_1:0',
  [1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell_2/kernel/RMSProp:0',
  [1132, 1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell_2/kernel/RMSProp_1:0',
  [1132, 1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell_2/bias/RMSProp:0',
  [1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell_2/bias/RMSProp_1:0',
  [1600]),
 ('rnn_style/gmm/weights/RMSProp:0', [400, 121]),
 ('rnn_style/gmm/weights/RMSProp_1:0', [400, 121]),
 ('rnn_style/gmm/biases/RMSProp:0', [121]),
 ('rnn_style/gmm/biases/RMSProp_1:0', [121])]
[[05/11/2025 09:13:08 AM]] trainable parameters:
[[05/11/2025 09:13:08 AM]] [('style_encoder/bidirectional/lstm_cell/kernel:0', [3, 512]),
 ('style_encoder/bidirectional/lstm_cell/recurrent_kernel:0', [128, 512]),
 ('style_encoder/bidirectional/lstm_cell/bias:0', [512]),
 ('style_encoder/bidirectional/lstm_cell_1/kernel:0', [3, 512]),
 ('style_encoder/bidirectional/lstm_cell_1/recurrent_kernel:0', [128, 512]),
 ('style_encoder/bidirectional/lstm_cell_1/bias:0', [512]),
 ('style_encoder/dense/kernel:0', [256, 1]),
 ('style_encoder/dense/bias:0', [1]),
 ('style_encoder/dense_1/kernel:0', [256, 256]),
 ('style_encoder/dense_1/bias:0', [256]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell/kernel:0', [732, 1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell/bias:0', [1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/attention/weights:0', [732, 30]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/attention/biases:0', [30]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell_1/kernel:0',
  [1132, 1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell_1/bias:0', [1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell_2/kernel:0',
  [1132, 1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell_2/bias:0', [1600]),
 ('rnn_style/gmm/weights:0', [400, 121]),
 ('rnn_style/gmm/biases:0', [121])]
[[05/11/2025 09:13:08 AM]] trainable parameter count:
[[05/11/2025 09:13:08 AM]] 5070128
[[05/11/2025 09:13:08 AM]] built graph
[[05/11/2025 09:13:08 AM]] Restoring parameters from final_checkpoints/model-10350
[[05/11/2025 09:13:08 AM]] Model initialized successfully
[[05/11/2025 09:13:19 AM]] Shutting down
[[05/11/2025 09:13:19 AM]] Waiting for application shutdown.
[[05/11/2025 09:13:19 AM]] Application shutdown complete.
[[05/11/2025 09:13:19 AM]] Finished server process [1249]
[[05/11/2025 09:13:22 AM]] 
new run with parameters:
{'attention_mixture_components': 10,
 'batch_size': 32,
 'batch_sizes': [32],
 'beta1_decay': 0.9,
 'beta1_decays': [0.9],
 'checkpoint_dir': 'checkpoints_style_synthesis',
 'early_stopping_steps': 2000,
 'enable_parameter_averaging': False,
 'grad_clip': 10,
 'keep_prob_scalar': 1.0,
 'learning_rate': 0.0001,
 'learning_rates': [0.0001],
 'log_dir': 'logs_style_synthesis',
 'log_interval': 50,
 'logging_level': 20,
 'loss_averaging_window': 100,
 'lstm_size': 400,
 'min_steps_to_checkpoint': 2000,
 'num_restarts': 0,
 'num_training_steps': 30000,
 'optimizer': 'rms',
 'output_mixture_components': 20,
 'output_units': 121,
 'patiences': [2000],
 'prediction_dir': 'predictions_style_synthesis',
 'reader': None,
 'regularization_constant': 0.0,
 'restart_idx': 0,
 'style_embedding_size': 256,
 'validation_batch_size': 32,
 'warm_start_init_step': 0}
[[05/11/2025 09:13:25 AM]] all parameters:
[[05/11/2025 09:13:25 AM]] [('Variable:0', []),
 ('Variable_1:0', []),
 ('Variable_2:0', []),
 ('style_encoder/bidirectional/lstm_cell/kernel:0', [3, 512]),
 ('style_encoder/bidirectional/lstm_cell/recurrent_kernel:0', [128, 512]),
 ('style_encoder/bidirectional/lstm_cell/bias:0', [512]),
 ('style_encoder/bidirectional/lstm_cell_1/kernel:0', [3, 512]),
 ('style_encoder/bidirectional/lstm_cell_1/recurrent_kernel:0', [128, 512]),
 ('style_encoder/bidirectional/lstm_cell_1/bias:0', [512]),
 ('style_encoder/dense/kernel:0', [256, 1]),
 ('style_encoder/dense/bias:0', [1]),
 ('style_encoder/dense_1/kernel:0', [256, 256]),
 ('style_encoder/dense_1/bias:0', [256]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell/kernel:0', [732, 1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell/bias:0', [1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/attention/weights:0', [732, 30]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/attention/biases:0', [30]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell_1/kernel:0',
  [1132, 1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell_1/bias:0', [1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell_2/kernel:0',
  [1132, 1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell_2/bias:0', [1600]),
 ('rnn_style/gmm/weights:0', [400, 121]),
 ('rnn_style/gmm/biases:0', [121]),
 ('style_encoder/bidirectional/lstm_cell/kernel/RMSProp:0', [3, 512]),
 ('style_encoder/bidirectional/lstm_cell/kernel/RMSProp_1:0', [3, 512]),
 ('style_encoder/bidirectional/lstm_cell/recurrent_kernel/RMSProp:0',
  [128, 512]),
 ('style_encoder/bidirectional/lstm_cell/recurrent_kernel/RMSProp_1:0',
  [128, 512]),
 ('style_encoder/bidirectional/lstm_cell/bias/RMSProp:0', [512]),
 ('style_encoder/bidirectional/lstm_cell/bias/RMSProp_1:0', [512]),
 ('style_encoder/bidirectional/lstm_cell_1/kernel/RMSProp:0', [3, 512]),
 ('style_encoder/bidirectional/lstm_cell_1/kernel/RMSProp_1:0', [3, 512]),
 ('style_encoder/bidirectional/lstm_cell_1/recurrent_kernel/RMSProp:0',
  [128, 512]),
 ('style_encoder/bidirectional/lstm_cell_1/recurrent_kernel/RMSProp_1:0',
  [128, 512]),
 ('style_encoder/bidirectional/lstm_cell_1/bias/RMSProp:0', [512]),
 ('style_encoder/bidirectional/lstm_cell_1/bias/RMSProp_1:0', [512]),
 ('style_encoder/dense/kernel/RMSProp:0', [256, 1]),
 ('style_encoder/dense/kernel/RMSProp_1:0', [256, 1]),
 ('style_encoder/dense/bias/RMSProp:0', [1]),
 ('style_encoder/dense/bias/RMSProp_1:0', [1]),
 ('style_encoder/dense_1/kernel/RMSProp:0', [256, 256]),
 ('style_encoder/dense_1/kernel/RMSProp_1:0', [256, 256]),
 ('style_encoder/dense_1/bias/RMSProp:0', [256]),
 ('style_encoder/dense_1/bias/RMSProp_1:0', [256]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell/kernel/RMSProp:0',
  [732, 1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell/kernel/RMSProp_1:0',
  [732, 1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell/bias/RMSProp:0', [1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell/bias/RMSProp_1:0',
  [1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/attention/weights/RMSProp:0',
  [732, 30]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/attention/weights/RMSProp_1:0',
  [732, 30]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/attention/biases/RMSProp:0', [30]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/attention/biases/RMSProp_1:0',
  [30]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell_1/kernel/RMSProp:0',
  [1132, 1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell_1/kernel/RMSProp_1:0',
  [1132, 1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell_1/bias/RMSProp:0',
  [1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell_1/bias/RMSProp_1:0',
  [1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell_2/kernel/RMSProp:0',
  [1132, 1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell_2/kernel/RMSProp_1:0',
  [1132, 1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell_2/bias/RMSProp:0',
  [1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell_2/bias/RMSProp_1:0',
  [1600]),
 ('rnn_style/gmm/weights/RMSProp:0', [400, 121]),
 ('rnn_style/gmm/weights/RMSProp_1:0', [400, 121]),
 ('rnn_style/gmm/biases/RMSProp:0', [121]),
 ('rnn_style/gmm/biases/RMSProp_1:0', [121])]
[[05/11/2025 09:13:25 AM]] trainable parameters:
[[05/11/2025 09:13:25 AM]] [('style_encoder/bidirectional/lstm_cell/kernel:0', [3, 512]),
 ('style_encoder/bidirectional/lstm_cell/recurrent_kernel:0', [128, 512]),
 ('style_encoder/bidirectional/lstm_cell/bias:0', [512]),
 ('style_encoder/bidirectional/lstm_cell_1/kernel:0', [3, 512]),
 ('style_encoder/bidirectional/lstm_cell_1/recurrent_kernel:0', [128, 512]),
 ('style_encoder/bidirectional/lstm_cell_1/bias:0', [512]),
 ('style_encoder/dense/kernel:0', [256, 1]),
 ('style_encoder/dense/bias:0', [1]),
 ('style_encoder/dense_1/kernel:0', [256, 256]),
 ('style_encoder/dense_1/bias:0', [256]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell/kernel:0', [732, 1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell/bias:0', [1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/attention/weights:0', [732, 30]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/attention/biases:0', [30]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell_1/kernel:0',
  [1132, 1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell_1/bias:0', [1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell_2/kernel:0',
  [1132, 1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell_2/bias:0', [1600]),
 ('rnn_style/gmm/weights:0', [400, 121]),
 ('rnn_style/gmm/biases:0', [121])]
[[05/11/2025 09:13:25 AM]] trainable parameter count:
[[05/11/2025 09:13:25 AM]] 5070128
[[05/11/2025 09:13:25 AM]] built graph
[[05/11/2025 09:13:25 AM]] Restoring parameters from final_checkpoints/model-10350
[[05/11/2025 09:13:25 AM]] Started server process [1394]
[[05/11/2025 09:13:25 AM]] Waiting for application startup.
[[05/11/2025 09:13:25 AM]] Application startup complete.
[[05/11/2025 09:13:27 AM]] Shutting down
[[05/11/2025 09:13:28 AM]] Waiting for application shutdown.
[[05/11/2025 09:13:28 AM]] Application shutdown complete.
[[05/11/2025 09:13:28 AM]] Finished server process [1394]
[[05/11/2025 09:13:30 AM]] 
new run with parameters:
{'attention_mixture_components': 10,
 'batch_size': 32,
 'batch_sizes': [32],
 'beta1_decay': 0.9,
 'beta1_decays': [0.9],
 'checkpoint_dir': 'checkpoints_style_synthesis',
 'early_stopping_steps': 2000,
 'enable_parameter_averaging': False,
 'grad_clip': 10,
 'keep_prob_scalar': 1.0,
 'learning_rate': 0.0001,
 'learning_rates': [0.0001],
 'log_dir': 'logs_style_synthesis',
 'log_interval': 50,
 'logging_level': 20,
 'loss_averaging_window': 100,
 'lstm_size': 400,
 'min_steps_to_checkpoint': 2000,
 'num_restarts': 0,
 'num_training_steps': 30000,
 'optimizer': 'rms',
 'output_mixture_components': 20,
 'output_units': 121,
 'patiences': [2000],
 'prediction_dir': 'predictions_style_synthesis',
 'reader': None,
 'regularization_constant': 0.0,
 'restart_idx': 0,
 'style_embedding_size': 256,
 'validation_batch_size': 32,
 'warm_start_init_step': 0}
[[05/11/2025 09:13:33 AM]] all parameters:
[[05/11/2025 09:13:33 AM]] [('Variable:0', []),
 ('Variable_1:0', []),
 ('Variable_2:0', []),
 ('style_encoder/bidirectional/lstm_cell/kernel:0', [3, 512]),
 ('style_encoder/bidirectional/lstm_cell/recurrent_kernel:0', [128, 512]),
 ('style_encoder/bidirectional/lstm_cell/bias:0', [512]),
 ('style_encoder/bidirectional/lstm_cell_1/kernel:0', [3, 512]),
 ('style_encoder/bidirectional/lstm_cell_1/recurrent_kernel:0', [128, 512]),
 ('style_encoder/bidirectional/lstm_cell_1/bias:0', [512]),
 ('style_encoder/dense/kernel:0', [256, 1]),
 ('style_encoder/dense/bias:0', [1]),
 ('style_encoder/dense_1/kernel:0', [256, 256]),
 ('style_encoder/dense_1/bias:0', [256]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell/kernel:0', [732, 1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell/bias:0', [1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/attention/weights:0', [732, 30]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/attention/biases:0', [30]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell_1/kernel:0',
  [1132, 1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell_1/bias:0', [1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell_2/kernel:0',
  [1132, 1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell_2/bias:0', [1600]),
 ('rnn_style/gmm/weights:0', [400, 121]),
 ('rnn_style/gmm/biases:0', [121]),
 ('style_encoder/bidirectional/lstm_cell/kernel/RMSProp:0', [3, 512]),
 ('style_encoder/bidirectional/lstm_cell/kernel/RMSProp_1:0', [3, 512]),
 ('style_encoder/bidirectional/lstm_cell/recurrent_kernel/RMSProp:0',
  [128, 512]),
 ('style_encoder/bidirectional/lstm_cell/recurrent_kernel/RMSProp_1:0',
  [128, 512]),
 ('style_encoder/bidirectional/lstm_cell/bias/RMSProp:0', [512]),
 ('style_encoder/bidirectional/lstm_cell/bias/RMSProp_1:0', [512]),
 ('style_encoder/bidirectional/lstm_cell_1/kernel/RMSProp:0', [3, 512]),
 ('style_encoder/bidirectional/lstm_cell_1/kernel/RMSProp_1:0', [3, 512]),
 ('style_encoder/bidirectional/lstm_cell_1/recurrent_kernel/RMSProp:0',
  [128, 512]),
 ('style_encoder/bidirectional/lstm_cell_1/recurrent_kernel/RMSProp_1:0',
  [128, 512]),
 ('style_encoder/bidirectional/lstm_cell_1/bias/RMSProp:0', [512]),
 ('style_encoder/bidirectional/lstm_cell_1/bias/RMSProp_1:0', [512]),
 ('style_encoder/dense/kernel/RMSProp:0', [256, 1]),
 ('style_encoder/dense/kernel/RMSProp_1:0', [256, 1]),
 ('style_encoder/dense/bias/RMSProp:0', [1]),
 ('style_encoder/dense/bias/RMSProp_1:0', [1]),
 ('style_encoder/dense_1/kernel/RMSProp:0', [256, 256]),
 ('style_encoder/dense_1/kernel/RMSProp_1:0', [256, 256]),
 ('style_encoder/dense_1/bias/RMSProp:0', [256]),
 ('style_encoder/dense_1/bias/RMSProp_1:0', [256]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell/kernel/RMSProp:0',
  [732, 1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell/kernel/RMSProp_1:0',
  [732, 1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell/bias/RMSProp:0', [1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell/bias/RMSProp_1:0',
  [1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/attention/weights/RMSProp:0',
  [732, 30]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/attention/weights/RMSProp_1:0',
  [732, 30]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/attention/biases/RMSProp:0', [30]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/attention/biases/RMSProp_1:0',
  [30]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell_1/kernel/RMSProp:0',
  [1132, 1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell_1/kernel/RMSProp_1:0',
  [1132, 1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell_1/bias/RMSProp:0',
  [1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell_1/bias/RMSProp_1:0',
  [1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell_2/kernel/RMSProp:0',
  [1132, 1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell_2/kernel/RMSProp_1:0',
  [1132, 1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell_2/bias/RMSProp:0',
  [1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell_2/bias/RMSProp_1:0',
  [1600]),
 ('rnn_style/gmm/weights/RMSProp:0', [400, 121]),
 ('rnn_style/gmm/weights/RMSProp_1:0', [400, 121]),
 ('rnn_style/gmm/biases/RMSProp:0', [121]),
 ('rnn_style/gmm/biases/RMSProp_1:0', [121])]
[[05/11/2025 09:13:33 AM]] trainable parameters:
[[05/11/2025 09:13:33 AM]] [('style_encoder/bidirectional/lstm_cell/kernel:0', [3, 512]),
 ('style_encoder/bidirectional/lstm_cell/recurrent_kernel:0', [128, 512]),
 ('style_encoder/bidirectional/lstm_cell/bias:0', [512]),
 ('style_encoder/bidirectional/lstm_cell_1/kernel:0', [3, 512]),
 ('style_encoder/bidirectional/lstm_cell_1/recurrent_kernel:0', [128, 512]),
 ('style_encoder/bidirectional/lstm_cell_1/bias:0', [512]),
 ('style_encoder/dense/kernel:0', [256, 1]),
 ('style_encoder/dense/bias:0', [1]),
 ('style_encoder/dense_1/kernel:0', [256, 256]),
 ('style_encoder/dense_1/bias:0', [256]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell/kernel:0', [732, 1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell/bias:0', [1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/attention/weights:0', [732, 30]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/attention/biases:0', [30]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell_1/kernel:0',
  [1132, 1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell_1/bias:0', [1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell_2/kernel:0',
  [1132, 1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell_2/bias:0', [1600]),
 ('rnn_style/gmm/weights:0', [400, 121]),
 ('rnn_style/gmm/biases:0', [121])]
[[05/11/2025 09:13:33 AM]] trainable parameter count:
[[05/11/2025 09:13:33 AM]] 5070128
[[05/11/2025 09:13:33 AM]] built graph
[[05/11/2025 09:13:33 AM]] Restoring parameters from final_checkpoints/model-10350
[[05/11/2025 09:13:33 AM]] Started server process [1527]
[[05/11/2025 09:13:33 AM]] Waiting for application startup.
[[05/11/2025 09:13:33 AM]] Application startup complete.
[[05/11/2025 09:13:35 AM]] 
new run with parameters:
{'attention_mixture_components': 10,
 'batch_size': 32,
 'batch_sizes': [32],
 'beta1_decay': 0.9,
 'beta1_decays': [0.9],
 'checkpoint_dir': 'checkpoints_style_synthesis',
 'early_stopping_steps': 2000,
 'enable_parameter_averaging': False,
 'grad_clip': 10,
 'keep_prob_scalar': 1.0,
 'learning_rate': 0.0001,
 'learning_rates': [0.0001],
 'log_dir': 'logs_style_synthesis',
 'log_interval': 50,
 'logging_level': 20,
 'loss_averaging_window': 100,
 'lstm_size': 400,
 'min_steps_to_checkpoint': 2000,
 'num_restarts': 0,
 'num_training_steps': 30000,
 'optimizer': 'rms',
 'output_mixture_components': 20,
 'output_units': 121,
 'patiences': [2000],
 'prediction_dir': 'predictions_style_synthesis',
 'reader': None,
 'regularization_constant': 0.0,
 'restart_idx': 0,
 'style_embedding_size': 256,
 'validation_batch_size': 32,
 'warm_start_init_step': 0}
[[05/11/2025 09:13:35 AM]] Received request with style_id: 3
[[05/11/2025 09:13:35 AM]] Markdown text length: 132
[[05/11/2025 09:13:35 AM]] 
new run with parameters:
{'attention_mixture_components': 10,
 'batch_size': 32,
 'batch_sizes': [32],
 'beta1_decay': 0.9,
 'beta1_decays': [0.9],
 'checkpoint_dir': 'checkpoints_style_synthesis',
 'early_stopping_steps': 2000,
 'enable_parameter_averaging': False,
 'grad_clip': 10,
 'keep_prob_scalar': 1.0,
 'learning_rate': 0.0001,
 'learning_rates': [0.0001],
 'log_dir': 'logs_style_synthesis',
 'log_interval': 50,
 'logging_level': 20,
 'loss_averaging_window': 100,
 'lstm_size': 400,
 'min_steps_to_checkpoint': 2000,
 'num_restarts': 0,
 'num_training_steps': 30000,
 'optimizer': 'rms',
 'output_mixture_components': 20,
 'output_units': 121,
 'patiences': [2000],
 'prediction_dir': 'predictions_style_synthesis',
 'reader': None,
 'regularization_constant': 0.0,
 'restart_idx': 0,
 'style_embedding_size': 256,
 'validation_batch_size': 32,
 'warm_start_init_step': 0}
[[05/11/2025 09:13:40 AM]] all parameters:
[[05/11/2025 09:13:40 AM]] [('Variable:0', []),
 ('Variable_1:0', []),
 ('Variable_2:0', []),
 ('style_encoder/bidirectional/lstm_cell/kernel:0', [3, 512]),
 ('style_encoder/bidirectional/lstm_cell/recurrent_kernel:0', [128, 512]),
 ('style_encoder/bidirectional/lstm_cell/bias:0', [512]),
 ('style_encoder/bidirectional/lstm_cell_1/kernel:0', [3, 512]),
 ('style_encoder/bidirectional/lstm_cell_1/recurrent_kernel:0', [128, 512]),
 ('style_encoder/bidirectional/lstm_cell_1/bias:0', [512]),
 ('style_encoder/dense/kernel:0', [256, 1]),
 ('style_encoder/dense/bias:0', [1]),
 ('style_encoder/dense_1/kernel:0', [256, 256]),
 ('style_encoder/dense_1/bias:0', [256]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell/kernel:0', [732, 1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell/bias:0', [1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/attention/weights:0', [732, 30]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/attention/biases:0', [30]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell_1/kernel:0',
  [1132, 1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell_1/bias:0', [1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell_2/kernel:0',
  [1132, 1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell_2/bias:0', [1600]),
 ('rnn_style/gmm/weights:0', [400, 121]),
 ('rnn_style/gmm/biases:0', [121]),
 ('style_encoder/bidirectional/lstm_cell/kernel/RMSProp:0', [3, 512]),
 ('style_encoder/bidirectional/lstm_cell/kernel/RMSProp_1:0', [3, 512]),
 ('style_encoder/bidirectional/lstm_cell/recurrent_kernel/RMSProp:0',
  [128, 512]),
 ('style_encoder/bidirectional/lstm_cell/recurrent_kernel/RMSProp_1:0',
  [128, 512]),
 ('style_encoder/bidirectional/lstm_cell/bias/RMSProp:0', [512]),
 ('style_encoder/bidirectional/lstm_cell/bias/RMSProp_1:0', [512]),
 ('style_encoder/bidirectional/lstm_cell_1/kernel/RMSProp:0', [3, 512]),
 ('style_encoder/bidirectional/lstm_cell_1/kernel/RMSProp_1:0', [3, 512]),
 ('style_encoder/bidirectional/lstm_cell_1/recurrent_kernel/RMSProp:0',
  [128, 512]),
 ('style_encoder/bidirectional/lstm_cell_1/recurrent_kernel/RMSProp_1:0',
  [128, 512]),
 ('style_encoder/bidirectional/lstm_cell_1/bias/RMSProp:0', [512]),
 ('style_encoder/bidirectional/lstm_cell_1/bias/RMSProp_1:0', [512]),
 ('style_encoder/dense/kernel/RMSProp:0', [256, 1]),
 ('style_encoder/dense/kernel/RMSProp_1:0', [256, 1]),
 ('style_encoder/dense/bias/RMSProp:0', [1]),
 ('style_encoder/dense/bias/RMSProp_1:0', [1]),
 ('style_encoder/dense_1/kernel/RMSProp:0', [256, 256]),
 ('style_encoder/dense_1/kernel/RMSProp_1:0', [256, 256]),
 ('style_encoder/dense_1/bias/RMSProp:0', [256]),
 ('style_encoder/dense_1/bias/RMSProp_1:0', [256]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell/kernel/RMSProp:0',
  [732, 1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell/kernel/RMSProp_1:0',
  [732, 1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell/bias/RMSProp:0', [1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell/bias/RMSProp_1:0',
  [1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/attention/weights/RMSProp:0',
  [732, 30]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/attention/weights/RMSProp_1:0',
  [732, 30]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/attention/biases/RMSProp:0', [30]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/attention/biases/RMSProp_1:0',
  [30]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell_1/kernel/RMSProp:0',
  [1132, 1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell_1/kernel/RMSProp_1:0',
  [1132, 1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell_1/bias/RMSProp:0',
  [1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell_1/bias/RMSProp_1:0',
  [1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell_2/kernel/RMSProp:0',
  [1132, 1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell_2/kernel/RMSProp_1:0',
  [1132, 1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell_2/bias/RMSProp:0',
  [1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell_2/bias/RMSProp_1:0',
  [1600]),
 ('rnn_style/gmm/weights/RMSProp:0', [400, 121]),
 ('rnn_style/gmm/weights/RMSProp_1:0', [400, 121]),
 ('rnn_style/gmm/biases/RMSProp:0', [121]),
 ('rnn_style/gmm/biases/RMSProp_1:0', [121])]
[[05/11/2025 09:13:40 AM]] trainable parameters:
[[05/11/2025 09:13:40 AM]] [('style_encoder/bidirectional/lstm_cell/kernel:0', [3, 512]),
 ('style_encoder/bidirectional/lstm_cell/recurrent_kernel:0', [128, 512]),
 ('style_encoder/bidirectional/lstm_cell/bias:0', [512]),
 ('style_encoder/bidirectional/lstm_cell_1/kernel:0', [3, 512]),
 ('style_encoder/bidirectional/lstm_cell_1/recurrent_kernel:0', [128, 512]),
 ('style_encoder/bidirectional/lstm_cell_1/bias:0', [512]),
 ('style_encoder/dense/kernel:0', [256, 1]),
 ('style_encoder/dense/bias:0', [1]),
 ('style_encoder/dense_1/kernel:0', [256, 256]),
 ('style_encoder/dense_1/bias:0', [256]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell/kernel:0', [732, 1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell/bias:0', [1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/attention/weights:0', [732, 30]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/attention/biases:0', [30]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell_1/kernel:0',
  [1132, 1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell_1/bias:0', [1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell_2/kernel:0',
  [1132, 1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell_2/bias:0', [1600]),
 ('rnn_style/gmm/weights:0', [400, 121]),
 ('rnn_style/gmm/biases:0', [121])]
[[05/11/2025 09:13:40 AM]] trainable parameter count:
[[05/11/2025 09:13:40 AM]] 5070128
[[05/11/2025 09:13:40 AM]] built graph
[[05/11/2025 09:13:40 AM]] all parameters:
[[05/11/2025 09:13:40 AM]] [('Variable:0', []),
 ('Variable_1:0', []),
 ('Variable_2:0', []),
 ('style_encoder/bidirectional/lstm_cell/kernel:0', [3, 512]),
 ('style_encoder/bidirectional/lstm_cell/recurrent_kernel:0', [128, 512]),
 ('style_encoder/bidirectional/lstm_cell/bias:0', [512]),
 ('style_encoder/bidirectional/lstm_cell_1/kernel:0', [3, 512]),
 ('style_encoder/bidirectional/lstm_cell_1/recurrent_kernel:0', [128, 512]),
 ('style_encoder/bidirectional/lstm_cell_1/bias:0', [512]),
 ('style_encoder/dense/kernel:0', [256, 1]),
 ('style_encoder/dense/bias:0', [1]),
 ('style_encoder/dense_1/kernel:0', [256, 256]),
 ('style_encoder/dense_1/bias:0', [256]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell/kernel:0', [732, 1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell/bias:0', [1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/attention/weights:0', [732, 30]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/attention/biases:0', [30]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell_1/kernel:0',
  [1132, 1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell_1/bias:0', [1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell_2/kernel:0',
  [1132, 1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell_2/bias:0', [1600]),
 ('rnn_style/gmm/weights:0', [400, 121]),
 ('rnn_style/gmm/biases:0', [121]),
 ('style_encoder/bidirectional/lstm_cell/kernel/RMSProp:0', [3, 512]),
 ('style_encoder/bidirectional/lstm_cell/kernel/RMSProp_1:0', [3, 512]),
 ('style_encoder/bidirectional/lstm_cell/recurrent_kernel/RMSProp:0',
  [128, 512]),
 ('style_encoder/bidirectional/lstm_cell/recurrent_kernel/RMSProp_1:0',
  [128, 512]),
 ('style_encoder/bidirectional/lstm_cell/bias/RMSProp:0', [512]),
 ('style_encoder/bidirectional/lstm_cell/bias/RMSProp_1:0', [512]),
 ('style_encoder/bidirectional/lstm_cell_1/kernel/RMSProp:0', [3, 512]),
 ('style_encoder/bidirectional/lstm_cell_1/kernel/RMSProp_1:0', [3, 512]),
 ('style_encoder/bidirectional/lstm_cell_1/recurrent_kernel/RMSProp:0',
  [128, 512]),
 ('style_encoder/bidirectional/lstm_cell_1/recurrent_kernel/RMSProp_1:0',
  [128, 512]),
 ('style_encoder/bidirectional/lstm_cell_1/bias/RMSProp:0', [512]),
 ('style_encoder/bidirectional/lstm_cell_1/bias/RMSProp_1:0', [512]),
 ('style_encoder/dense/kernel/RMSProp:0', [256, 1]),
 ('style_encoder/dense/kernel/RMSProp_1:0', [256, 1]),
 ('style_encoder/dense/bias/RMSProp:0', [1]),
 ('style_encoder/dense/bias/RMSProp_1:0', [1]),
 ('style_encoder/dense_1/kernel/RMSProp:0', [256, 256]),
 ('style_encoder/dense_1/kernel/RMSProp_1:0', [256, 256]),
 ('style_encoder/dense_1/bias/RMSProp:0', [256]),
 ('style_encoder/dense_1/bias/RMSProp_1:0', [256]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell/kernel/RMSProp:0',
  [732, 1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell/kernel/RMSProp_1:0',
  [732, 1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell/bias/RMSProp:0', [1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell/bias/RMSProp_1:0',
  [1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/attention/weights/RMSProp:0',
  [732, 30]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/attention/weights/RMSProp_1:0',
  [732, 30]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/attention/biases/RMSProp:0', [30]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/attention/biases/RMSProp_1:0',
  [30]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell_1/kernel/RMSProp:0',
  [1132, 1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell_1/kernel/RMSProp_1:0',
  [1132, 1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell_1/bias/RMSProp:0',
  [1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell_1/bias/RMSProp_1:0',
  [1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell_2/kernel/RMSProp:0',
  [1132, 1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell_2/kernel/RMSProp_1:0',
  [1132, 1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell_2/bias/RMSProp:0',
  [1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell_2/bias/RMSProp_1:0',
  [1600]),
 ('rnn_style/gmm/weights/RMSProp:0', [400, 121]),
 ('rnn_style/gmm/weights/RMSProp_1:0', [400, 121]),
 ('rnn_style/gmm/biases/RMSProp:0', [121]),
 ('rnn_style/gmm/biases/RMSProp_1:0', [121])]
[[05/11/2025 09:13:40 AM]] trainable parameters:
[[05/11/2025 09:13:40 AM]] [('style_encoder/bidirectional/lstm_cell/kernel:0', [3, 512]),
 ('style_encoder/bidirectional/lstm_cell/recurrent_kernel:0', [128, 512]),
 ('style_encoder/bidirectional/lstm_cell/bias:0', [512]),
 ('style_encoder/bidirectional/lstm_cell_1/kernel:0', [3, 512]),
 ('style_encoder/bidirectional/lstm_cell_1/recurrent_kernel:0', [128, 512]),
 ('style_encoder/bidirectional/lstm_cell_1/bias:0', [512]),
 ('style_encoder/dense/kernel:0', [256, 1]),
 ('style_encoder/dense/bias:0', [1]),
 ('style_encoder/dense_1/kernel:0', [256, 256]),
 ('style_encoder/dense_1/bias:0', [256]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell/kernel:0', [732, 1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell/bias:0', [1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/attention/weights:0', [732, 30]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/attention/biases:0', [30]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell_1/kernel:0',
  [1132, 1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell_1/bias:0', [1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell_2/kernel:0',
  [1132, 1600]),
 ('rnn_style/StyleAdaptiveLSTMAttentionCell/lstm_cell_2/bias:0', [1600]),
 ('rnn_style/gmm/weights:0', [400, 121]),
 ('rnn_style/gmm/biases:0', [121])]
[[05/11/2025 09:13:40 AM]] trainable parameter count:
[[05/11/2025 09:13:40 AM]] 5070128
[[05/11/2025 09:13:40 AM]] Restoring parameters from final_checkpoints/model-10350
[[05/11/2025 09:13:40 AM]] built graph
[[05/11/2025 09:13:40 AM]] Restoring parameters from final_checkpoints/model-10350
[[05/11/2025 09:13:41 AM]] Model initialized successfully
[[05/11/2025 09:13:41 AM]] Model initialized successfully
[[05/11/2025 09:13:41 AM]] Saved request data to saved_requests/request_20250511_091341_4f0ca920.json
[[05/11/2025 09:13:41 AM]] Parsed 15 lines from markdown
[[05/11/2025 09:13:41 AM]] Line 0: 'Meeting Notes' (type: header)
[[05/11/2025 09:13:41 AM]] Line 1: '' (type: paragraph)
[[05/11/2025 09:13:41 AM]] Line 2: 'Attendees' (type: header)
[[05/11/2025 09:13:41 AM]] Line 3: 'List of' (type: bullet)
[[05/11/2025 09:13:41 AM]] Line 4: 'attendees' (type: bullet)
[[05/11/2025 09:13:41 AM]] Line 5: 'not' (type: bullet)
[[05/11/2025 09:13:41 AM]] Line 6: 'provided' (type: bullet)
[[05/11/2025 09:13:41 AM]] Line 7: '' (type: paragraph)
[[05/11/2025 09:13:41 AM]] Line 8: 'Agenda' (type: header)
[[05/11/2025 09:13:41 AM]] Line 9: 'Agenda not' (type: bullet)
[[05/11/2025 09:13:41 AM]] Line 10: 'provided' (type: bullet)
[[05/11/2025 09:13:41 AM]] Line 11: '' (type: paragraph)
[[05/11/2025 09:13:41 AM]] Line 12: 'Notes' (type: header)
[[05/11/2025 09:13:41 AM]] Line 13: 'Notes not' (type: bullet)
[[05/11/2025 09:13:41 AM]] Line 14: 'provided' (type: bullet)
[[05/11/2025 09:13:41 AM]] Using 16 worker threads with 32 CPU cores
[[05/11/2025 09:13:41 AM]] Processing item 0 with text: 'Meeting Notes'
[[05/11/2025 09:13:41 AM]] Text length: 13
[[05/11/2025 09:13:41 AM]] Processing item 1 with text: ''
[[05/11/2025 09:13:41 AM]] Text length: 1
[[05/11/2025 09:13:41 AM]] Processing item 2 with text: 'Attendees'
[[05/11/2025 09:13:41 AM]] Loaded style 3 with 589 strokes and 17 chars
[[05/11/2025 09:13:41 AM]] Processing item 3 with text: 'List of'
[[05/11/2025 09:13:41 AM]] Text length: 9
[[05/11/2025 09:13:41 AM]] Processing item 4 with text: 'attendees'
[[05/11/2025 09:13:41 AM]] Processing text: ''
[[05/11/2025 09:13:41 AM]] Text length: 7
[[05/11/2025 09:13:41 AM]] Loaded style 3 with 589 strokes and 17 chars
[[05/11/2025 09:13:41 AM]] Processing text: 'Meeting Notes'
[[05/11/2025 09:13:41 AM]] Processing item 5 with text: 'not'
[[05/11/2025 09:13:41 AM]] Encoded character sequence length: 19
[[05/11/2025 09:13:41 AM]] Processing item 6 with text: 'provided'
[[05/11/2025 09:13:41 AM]] Loaded style 3 with 589 strokes and 17 chars
[[05/11/2025 09:13:41 AM]] Processing item 7 with text: ''
[[05/11/2025 09:13:41 AM]] Text length: 9
[[05/11/2025 09:13:41 AM]] Encoded character sequence length: 32
[[05/11/2025 09:13:41 AM]] Text length: 3
[[05/11/2025 09:13:41 AM]] Processing item 8 with text: 'Agenda'
[[05/11/2025 09:13:41 AM]] Loaded style 3 with 589 strokes and 17 chars
[[05/11/2025 09:13:41 AM]] Created tensors - x_prime shape: (1, 1200, 3), chars shape: (1, 120), max_tsteps: 40
[[05/11/2025 09:13:41 AM]] Processing item 9 with text: 'Agenda not'
[[05/11/2025 09:13:41 AM]] Text length: 8
[[05/11/2025 09:13:41 AM]] Processing text: 'Attendees'
[[05/11/2025 09:13:41 AM]] Text length: 1
[[05/11/2025 09:13:41 AM]] Processing item 10 with text: 'provided'
[[05/11/2025 09:13:41 AM]] Created tensors - x_prime shape: (1, 1200, 3), chars shape: (1, 120), max_tsteps: 520
[[05/11/2025 09:13:41 AM]] Text length: 6
[[05/11/2025 09:13:41 AM]] Processing item 11 with text: ''
[[05/11/2025 09:13:41 AM]] Processing text: 'List of'
[[05/11/2025 09:13:41 AM]] Processing item 12 with text: 'Notes'
[[05/11/2025 09:13:41 AM]] x_prime shape: (1, 1200, 3)
[[05/11/2025 09:13:41 AM]] Loaded style 3 with 589 strokes and 17 chars
[[05/11/2025 09:13:41 AM]] Text length: 10
[[05/11/2025 09:13:41 AM]] Processing item 13 with text: 'Notes not'
[[05/11/2025 09:13:41 AM]] Encoded character sequence length: 28
[[05/11/2025 09:13:41 AM]] Loaded style 3 with 589 strokes and 17 chars
[[05/11/2025 09:13:41 AM]] Processing item 14 with text: 'provided'
[[05/11/2025 09:13:41 AM]] Text length: 8
[[05/11/2025 09:13:41 AM]] Loaded style 3 with 589 strokes and 17 chars
[[05/11/2025 09:13:41 AM]] x_prime shape: (1, 1200, 3)
[[05/11/2025 09:13:41 AM]] Text length: 1
[[05/11/2025 09:13:41 AM]] Loaded style 3 with 589 strokes and 17 chars
[[05/11/2025 09:13:41 AM]] Encoded character sequence length: 26
[[05/11/2025 09:13:41 AM]] Text length: 5
[[05/11/2025 09:13:41 AM]] x_prime_len: [589]
[[05/11/2025 09:13:41 AM]] Loaded style 3 with 589 strokes and 17 chars
[[05/11/2025 09:13:41 AM]] Processing text: 'attendees'
[[05/11/2025 09:13:41 AM]] Text length: 9
[[05/11/2025 09:13:41 AM]] Created tensors - x_prime shape: (1, 1200, 3), chars shape: (1, 120), max_tsteps: 360
[[05/11/2025 09:13:41 AM]] Processing text: 'provided'
[[05/11/2025 09:13:41 AM]] Text length: 8
[[05/11/2025 09:13:41 AM]] Loaded style 3 with 589 strokes and 17 chars
[[05/11/2025 09:13:41 AM]] Processing text: 'not'
[[05/11/2025 09:13:41 AM]] x_prime_len: [589]
[[05/11/2025 09:13:41 AM]] Processing text: ''
[[05/11/2025 09:13:41 AM]] Created tensors - x_prime shape: (1, 1200, 3), chars shape: (1, 120), max_tsteps: 280
[[05/11/2025 09:13:41 AM]] chars shape: (1, 120)
[[05/11/2025 09:13:41 AM]] Loaded style 3 with 589 strokes and 17 chars
[[05/11/2025 09:13:41 AM]] Processing text: 'Agenda'
[[05/11/2025 09:13:41 AM]] Loaded style 3 with 589 strokes and 17 chars
[[05/11/2025 09:13:41 AM]] Loaded style 3 with 589 strokes and 17 chars
[[05/11/2025 09:13:41 AM]] Encoded character sequence length: 28
[[05/11/2025 09:13:41 AM]] x_prime shape: (1, 1200, 3)
[[05/11/2025 09:13:41 AM]] Loaded style 3 with 589 strokes and 17 chars
[[05/11/2025 09:13:41 AM]] Encoded character sequence length: 27
[[05/11/2025 09:13:41 AM]] Processing text: 'Agenda not'
[[05/11/2025 09:13:41 AM]] Encoded character sequence length: 22
[[05/11/2025 09:13:41 AM]] chars shape: (1, 120)
[[05/11/2025 09:13:41 AM]] Loaded style 3 with 589 strokes and 17 chars
[[05/11/2025 09:13:41 AM]] Encoded character sequence length: 19
[[05/11/2025 09:13:41 AM]] x_prime shape: (1, 1200, 3)
[[05/11/2025 09:13:41 AM]] chars_len: [19]
[[05/11/2025 09:13:41 AM]] Processing text: 'provided'
[[05/11/2025 09:13:41 AM]] Encoded character sequence length: 25
[[05/11/2025 09:13:41 AM]] Processing text: ''
[[05/11/2025 09:13:41 AM]] Processing text: 'Notes'
[[05/11/2025 09:13:41 AM]] Created tensors - x_prime shape: (1, 1200, 3), chars shape: (1, 120), max_tsteps: 360
[[05/11/2025 09:13:41 AM]] x_prime_len: [589]
[[05/11/2025 09:13:41 AM]] Processing text: 'Notes not'
[[05/11/2025 09:13:41 AM]] Created tensors - x_prime shape: (1, 1200, 3), chars shape: (1, 120), max_tsteps: 320
[[05/11/2025 09:13:41 AM]] Encoded character sequence length: 29
[[05/11/2025 09:13:41 AM]] chars_len: [32]
[[05/11/2025 09:13:41 AM]] Created tensors - x_prime shape: (1, 1200, 3), chars shape: (1, 120), max_tsteps: 120
[[05/11/2025 09:13:41 AM]] Processing text: 'provided'
[[05/11/2025 09:13:41 AM]] x_prime_len: [589]
[[05/11/2025 09:13:41 AM]] max_tsteps: 40
[[05/11/2025 09:13:41 AM]] Created tensors - x_prime shape: (1, 1200, 3), chars shape: (1, 120), max_tsteps: 40
[[05/11/2025 09:13:41 AM]] Encoded character sequence length: 27
[[05/11/2025 09:13:41 AM]] Created tensors - x_prime shape: (1, 1200, 3), chars shape: (1, 120), max_tsteps: 240
[[05/11/2025 09:13:41 AM]] Encoded character sequence length: 19
[[05/11/2025 09:13:41 AM]] Encoded character sequence length: 24
[[05/11/2025 09:13:41 AM]] x_prime shape: (1, 1200, 3)
[[05/11/2025 09:13:41 AM]] chars shape: (1, 120)
[[05/11/2025 09:13:41 AM]] Encoded character sequence length: 28
[[05/11/2025 09:13:41 AM]] x_prime shape: (1, 1200, 3)
[[05/11/2025 09:13:41 AM]] max_tsteps: 520
[[05/11/2025 09:13:41 AM]] Created tensors - x_prime shape: (1, 1200, 3), chars shape: (1, 120), max_tsteps: 400
[[05/11/2025 09:13:41 AM]] x_prime shape: (1, 1200, 3)
[[05/11/2025 09:13:41 AM]] Encoded character sequence length: 27
[[05/11/2025 09:13:41 AM]] chars shape: (1, 120)
[[05/11/2025 09:13:41 AM]] Using reference strokes with shape: (0,)
[[05/11/2025 09:13:41 AM]] x_prime shape: (1, 1200, 3)
[[05/11/2025 09:13:41 AM]] x_prime shape: (1, 1200, 3)
[[05/11/2025 09:13:41 AM]] Created tensors - x_prime shape: (1, 1200, 3), chars shape: (1, 120), max_tsteps: 320
[[05/11/2025 09:13:41 AM]] x_prime_len: [589]
[[05/11/2025 09:13:41 AM]] Created tensors - x_prime shape: (1, 1200, 3), chars shape: (1, 120), max_tsteps: 200
[[05/11/2025 09:13:41 AM]] chars_len: [28]
[[05/11/2025 09:13:41 AM]] Created tensors - x_prime shape: (1, 1200, 3), chars shape: (1, 120), max_tsteps: 40
[[05/11/2025 09:13:41 AM]] Created tensors - x_prime shape: (1, 1200, 3), chars shape: (1, 120), max_tsteps: 360
[[05/11/2025 09:13:41 AM]] x_prime_len: [589]
[[05/11/2025 09:13:41 AM]] Using reference strokes with shape: (0,)
[[05/11/2025 09:13:41 AM]] x_prime shape: (1, 1200, 3)
[[05/11/2025 09:13:41 AM]] x_prime_len: [589]
[[05/11/2025 09:13:41 AM]] chars_len: [26]
[[05/11/2025 09:13:41 AM]] Created tensors - x_prime shape: (1, 1200, 3), chars shape: (1, 120), max_tsteps: 320
[[05/11/2025 09:13:41 AM]] x_prime_len: [589]
[[05/11/2025 09:13:41 AM]] x_prime_len: [589]
[[05/11/2025 09:13:41 AM]] x_prime shape: (1, 1200, 3)
[[05/11/2025 09:13:41 AM]] chars shape: (1, 120)
[[05/11/2025 09:13:41 AM]] x_prime shape: (1, 1200, 3)
[[05/11/2025 09:13:41 AM]] max_tsteps: 360
[[05/11/2025 09:13:41 AM]] x_prime shape: (1, 1200, 3)
[[05/11/2025 09:13:41 AM]] x_prime shape: (1, 1200, 3)
[[05/11/2025 09:13:41 AM]] chars shape: (1, 120)
[[05/11/2025 09:13:41 AM]] x_prime_len: [589]
[[05/11/2025 09:13:41 AM]] chars shape: (1, 120)
[[05/11/2025 09:13:41 AM]] max_tsteps: 280
[[05/11/2025 09:13:41 AM]] x_prime shape: (1, 1200, 3)
[[05/11/2025 09:13:41 AM]] chars shape: (1, 120)
[[05/11/2025 09:13:41 AM]] chars shape: (1, 120)
[[05/11/2025 09:13:41 AM]] x_prime_len: [589]
[[05/11/2025 09:13:41 AM]] chars_len: [28]
[[05/11/2025 09:13:41 AM]] x_prime_len: [589]
[[05/11/2025 09:13:41 AM]] Using reference strokes with shape: (0,)
[[05/11/2025 09:13:41 AM]] x_prime_len: [589]
[[05/11/2025 09:13:41 AM]] x_prime_len: [589]
[[05/11/2025 09:13:41 AM]] chars_len: [27]
[[05/11/2025 09:13:41 AM]] chars shape: (1, 120)
[[05/11/2025 09:13:41 AM]] chars_len: [22]
[[05/11/2025 09:13:41 AM]] Using reference strokes with shape: (0,)
[[05/11/2025 09:13:41 AM]] x_prime_len: [589]
[[05/11/2025 09:13:41 AM]] chars_len: [19]
[[05/11/2025 09:13:41 AM]] chars_len: [25]
[[05/11/2025 09:13:41 AM]] chars shape: (1, 120)
[[05/11/2025 09:13:41 AM]] max_tsteps: 360
[[05/11/2025 09:13:41 AM]] chars shape: (1, 120)
[[05/11/2025 09:13:41 AM]] chars shape: (1, 120)
[[05/11/2025 09:13:41 AM]] chars shape: (1, 120)
[[05/11/2025 09:13:41 AM]] max_tsteps: 320
[[05/11/2025 09:13:41 AM]] chars_len: [29]
[[05/11/2025 09:13:41 AM]] max_tsteps: 120
[[05/11/2025 09:13:41 AM]] chars shape: (1, 120)
[[05/11/2025 09:13:41 AM]] max_tsteps: 40
[[05/11/2025 09:13:41 AM]] max_tsteps: 240
[[05/11/2025 09:13:41 AM]] chars_len: [27]
[[05/11/2025 09:13:41 AM]] Using reference strokes with shape: (0,)
[[05/11/2025 09:13:41 AM]] chars_len: [24]
[[05/11/2025 09:13:41 AM]] chars_len: [19]
[[05/11/2025 09:13:41 AM]] chars_len: [28]
[[05/11/2025 09:13:41 AM]] Using reference strokes with shape: (0,)
[[05/11/2025 09:13:41 AM]] max_tsteps: 400
[[05/11/2025 09:13:41 AM]] Using reference strokes with shape: (0,)
[[05/11/2025 09:13:41 AM]] chars_len: [27]
[[05/11/2025 09:13:41 AM]] Using reference strokes with shape: (0,)
[[05/11/2025 09:13:41 AM]] Using reference strokes with shape: (0,)
[[05/11/2025 09:13:41 AM]] max_tsteps: 320
[[05/11/2025 09:13:41 AM]] max_tsteps: 200
[[05/11/2025 09:13:41 AM]] max_tsteps: 40
[[05/11/2025 09:13:41 AM]] max_tsteps: 360
[[05/11/2025 09:13:41 AM]] Using reference strokes with shape: (0,)
[[05/11/2025 09:13:41 AM]] max_tsteps: 320
[[05/11/2025 09:13:41 AM]] Using reference strokes with shape: (0,)
[[05/11/2025 09:13:41 AM]] Using reference strokes with shape: (0,)
[[05/11/2025 09:13:41 AM]] Using reference strokes with shape: (0,)
[[05/11/2025 09:13:41 AM]] Using reference strokes with shape: (0,)
[[05/11/2025 09:13:41 AM]] Using reference strokes with shape: (0,)
[[05/11/2025 09:13:41 AM]] Error during model inference: TensorArray has size zero, but element shape [?,128] is not fully defined. Currently only static shapes are supported when packing zero-size TensorArrays.
	 [[Node: style_encoder/bidirectional/TensorArrayStack/TensorArrayGatherV3 = TensorArrayGatherV3[_class=["loc:@style_encoder/bidirectional/TensorArray"], dtype=DT_FLOAT, element_shape=[?,128], _device="/job:localhost/replica:0/task:0/device:CPU:0"](style_encoder/bidirectional/TensorArray, style_encoder/bidirectional/TensorArrayStack/range, style_encoder/bidirectional/while/Exit_1)]]

Caused by op 'style_encoder/bidirectional/TensorArrayStack/TensorArrayGatherV3', defined at:
  File "/usr/local/lib/python3.6/threading.py", line 884, in _bootstrap
    self._bootstrap_inner()
  File "/usr/local/lib/python3.6/threading.py", line 916, in _bootstrap_inner
    self.run()
  File "/usr/local/lib/python3.6/site-packages/anyio/_backends/_asyncio.py", line 867, in run
    result = context.run(func, *args)
  File "/usr/local/lib/python3.6/site-packages/contextvars/__init__.py", line 38, in run
    return callable(*args, **kwargs)
  File "./main.py", line 616, in convert_markdown
    initialize_model()
  File "./main.py", line 78, in initialize_model
    style_embedding_size=256
  File "./train.py", line 56, in __init__
    super(StyleSynthesisModel, self).__init__(reader=None, **kwargs)
  File "./tf_base_model.py", line 110, in __init__
    self.graph = self.build_graph()
  File "./tf_base_model.py", line 401, in build_graph
    self.loss = self.calculate_loss()
  File "./train.py", line 81, in calculate_loss
    style_emb = style_encoder(self.ref_x)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/engine/topology.py", line 258, in __call__
    output = super(Layer, self).__call__(inputs, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/layers/base.py", line 696, in __call__
    outputs = self.call(inputs, *args, **kwargs)
  File "./train.py", line 26, in call
    x = self.bi_rnn(stroke_sequence)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/engine/topology.py", line 258, in __call__
    output = super(Layer, self).__call__(inputs, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/layers/base.py", line 696, in __call__
    outputs = self.call(inputs, *args, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/layers/wrappers.py", line 346, in call
    y = self.forward_layer.call(inputs, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/layers/recurrent.py", line 2056, in call
    inputs, mask=mask, training=training, initial_state=initial_state)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/layers/recurrent.py", line 613, in call
    input_length=timesteps)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/backend.py", line 2963, in rnn
    outputs = output_ta.stack()
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/ops/tensor_array_ops.py", line 893, in stack
    return self._implementation.stack(name=name)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/ops/tensor_array_ops.py", line 291, in stack
    return self.gather(math_ops.range(0, self.size()), name=name)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/ops/tensor_array_ops.py", line 305, in gather
    element_shape=element_shape)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/ops/gen_data_flow_ops.py", line 4186, in _tensor_array_gather_v3
    flow_in=flow_in, dtype=dtype, element_shape=element_shape, name=name)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/framework/op_def_library.py", line 787, in _apply_op_helper
    op_def=op_def)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/framework/ops.py", line 3271, in create_op
    op_def=op_def)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/framework/ops.py", line 1650, in __init__
    self._traceback = self._graph._extract_stack()  # pylint: disable=protected-access

UnimplementedError (see above for traceback): TensorArray has size zero, but element shape [?,128] is not fully defined. Currently only static shapes are supported when packing zero-size TensorArrays.
	 [[Node: style_encoder/bidirectional/TensorArrayStack/TensorArrayGatherV3 = TensorArrayGatherV3[_class=["loc:@style_encoder/bidirectional/TensorArray"], dtype=DT_FLOAT, element_shape=[?,128], _device="/job:localhost/replica:0/task:0/device:CPU:0"](style_encoder/bidirectional/TensorArray, style_encoder/bidirectional/TensorArrayStack/range, style_encoder/bidirectional/while/Exit_1)]]

[[05/11/2025 09:13:41 AM]] Processing failed for item 1: TensorArray has size zero, but element shape [?,128] is not fully defined. Currently only static shapes are supported when packing zero-size TensorArrays.
	 [[Node: style_encoder/bidirectional/TensorArrayStack/TensorArrayGatherV3 = TensorArrayGatherV3[_class=["loc:@style_encoder/bidirectional/TensorArray"], dtype=DT_FLOAT, element_shape=[?,128], _device="/job:localhost/replica:0/task:0/device:CPU:0"](style_encoder/bidirectional/TensorArray, style_encoder/bidirectional/TensorArrayStack/range, style_encoder/bidirectional/while/Exit_1)]]

Caused by op 'style_encoder/bidirectional/TensorArrayStack/TensorArrayGatherV3', defined at:
  File "/usr/local/lib/python3.6/threading.py", line 884, in _bootstrap
    self._bootstrap_inner()
  File "/usr/local/lib/python3.6/threading.py", line 916, in _bootstrap_inner
    self.run()
  File "/usr/local/lib/python3.6/site-packages/anyio/_backends/_asyncio.py", line 867, in run
    result = context.run(func, *args)
  File "/usr/local/lib/python3.6/site-packages/contextvars/__init__.py", line 38, in run
    return callable(*args, **kwargs)
  File "./main.py", line 616, in convert_markdown
    initialize_model()
  File "./main.py", line 78, in initialize_model
    style_embedding_size=256
  File "./train.py", line 56, in __init__
    super(StyleSynthesisModel, self).__init__(reader=None, **kwargs)
  File "./tf_base_model.py", line 110, in __init__
    self.graph = self.build_graph()
  File "./tf_base_model.py", line 401, in build_graph
    self.loss = self.calculate_loss()
  File "./train.py", line 81, in calculate_loss
    style_emb = style_encoder(self.ref_x)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/engine/topology.py", line 258, in __call__
    output = super(Layer, self).__call__(inputs, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/layers/base.py", line 696, in __call__
    outputs = self.call(inputs, *args, **kwargs)
  File "./train.py", line 26, in call
    x = self.bi_rnn(stroke_sequence)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/engine/topology.py", line 258, in __call__
    output = super(Layer, self).__call__(inputs, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/layers/base.py", line 696, in __call__
    outputs = self.call(inputs, *args, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/layers/wrappers.py", line 346, in call
    y = self.forward_layer.call(inputs, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/layers/recurrent.py", line 2056, in call
    inputs, mask=mask, training=training, initial_state=initial_state)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/layers/recurrent.py", line 613, in call
    input_length=timesteps)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/backend.py", line 2963, in rnn
    outputs = output_ta.stack()
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/ops/tensor_array_ops.py", line 893, in stack
    return self._implementation.stack(name=name)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/ops/tensor_array_ops.py", line 291, in stack
    return self.gather(math_ops.range(0, self.size()), name=name)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/ops/tensor_array_ops.py", line 305, in gather
    element_shape=element_shape)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/ops/gen_data_flow_ops.py", line 4186, in _tensor_array_gather_v3
    flow_in=flow_in, dtype=dtype, element_shape=element_shape, name=name)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/framework/op_def_library.py", line 787, in _apply_op_helper
    op_def=op_def)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/framework/ops.py", line 3271, in create_op
    op_def=op_def)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/framework/ops.py", line 1650, in __init__
    self._traceback = self._graph._extract_stack()  # pylint: disable=protected-access

UnimplementedError (see above for traceback): TensorArray has size zero, but element shape [?,128] is not fully defined. Currently only static shapes are supported when packing zero-size TensorArrays.
	 [[Node: style_encoder/bidirectional/TensorArrayStack/TensorArrayGatherV3 = TensorArrayGatherV3[_class=["loc:@style_encoder/bidirectional/TensorArray"], dtype=DT_FLOAT, element_shape=[?,128], _device="/job:localhost/replica:0/task:0/device:CPU:0"](style_encoder/bidirectional/TensorArray, style_encoder/bidirectional/TensorArrayStack/range, style_encoder/bidirectional/while/Exit_1)]]

[[05/11/2025 09:13:41 AM]] Error during model inference: TensorArray has size zero, but element shape [?,128] is not fully defined. Currently only static shapes are supported when packing zero-size TensorArrays.
	 [[Node: style_encoder/bidirectional/TensorArrayStack/TensorArrayGatherV3 = TensorArrayGatherV3[_class=["loc:@style_encoder/bidirectional/TensorArray"], dtype=DT_FLOAT, element_shape=[?,128], _device="/job:localhost/replica:0/task:0/device:CPU:0"](style_encoder/bidirectional/TensorArray, style_encoder/bidirectional/TensorArrayStack/range, style_encoder/bidirectional/while/Exit_1)]]

Caused by op 'style_encoder/bidirectional/TensorArrayStack/TensorArrayGatherV3', defined at:
  File "/usr/local/lib/python3.6/threading.py", line 884, in _bootstrap
    self._bootstrap_inner()
  File "/usr/local/lib/python3.6/threading.py", line 916, in _bootstrap_inner
    self.run()
  File "/usr/local/lib/python3.6/site-packages/anyio/_backends/_asyncio.py", line 867, in run
    result = context.run(func, *args)
  File "/usr/local/lib/python3.6/site-packages/contextvars/__init__.py", line 38, in run
    return callable(*args, **kwargs)
  File "./main.py", line 616, in convert_markdown
    initialize_model()
  File "./main.py", line 78, in initialize_model
    style_embedding_size=256
  File "./train.py", line 56, in __init__
    super(StyleSynthesisModel, self).__init__(reader=None, **kwargs)
  File "./tf_base_model.py", line 110, in __init__
    self.graph = self.build_graph()
  File "./tf_base_model.py", line 401, in build_graph
    self.loss = self.calculate_loss()
  File "./train.py", line 81, in calculate_loss
    style_emb = style_encoder(self.ref_x)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/engine/topology.py", line 258, in __call__
    output = super(Layer, self).__call__(inputs, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/layers/base.py", line 696, in __call__
    outputs = self.call(inputs, *args, **kwargs)
  File "./train.py", line 26, in call
    x = self.bi_rnn(stroke_sequence)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/engine/topology.py", line 258, in __call__
    output = super(Layer, self).__call__(inputs, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/layers/base.py", line 696, in __call__
    outputs = self.call(inputs, *args, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/layers/wrappers.py", line 346, in call
    y = self.forward_layer.call(inputs, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/layers/recurrent.py", line 2056, in call
    inputs, mask=mask, training=training, initial_state=initial_state)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/layers/recurrent.py", line 613, in call
    input_length=timesteps)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/backend.py", line 2963, in rnn
    outputs = output_ta.stack()
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/ops/tensor_array_ops.py", line 893, in stack
    return self._implementation.stack(name=name)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/ops/tensor_array_ops.py", line 291, in stack
    return self.gather(math_ops.range(0, self.size()), name=name)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/ops/tensor_array_ops.py", line 305, in gather
    element_shape=element_shape)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/ops/gen_data_flow_ops.py", line 4186, in _tensor_array_gather_v3
    flow_in=flow_in, dtype=dtype, element_shape=element_shape, name=name)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/framework/op_def_library.py", line 787, in _apply_op_helper
    op_def=op_def)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/framework/ops.py", line 3271, in create_op
    op_def=op_def)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/framework/ops.py", line 1650, in __init__
    self._traceback = self._graph._extract_stack()  # pylint: disable=protected-access

UnimplementedError (see above for traceback): TensorArray has size zero, but element shape [?,128] is not fully defined. Currently only static shapes are supported when packing zero-size TensorArrays.
	 [[Node: style_encoder/bidirectional/TensorArrayStack/TensorArrayGatherV3 = TensorArrayGatherV3[_class=["loc:@style_encoder/bidirectional/TensorArray"], dtype=DT_FLOAT, element_shape=[?,128], _device="/job:localhost/replica:0/task:0/device:CPU:0"](style_encoder/bidirectional/TensorArray, style_encoder/bidirectional/TensorArrayStack/range, style_encoder/bidirectional/while/Exit_1)]]

[[05/11/2025 09:13:41 AM]] Processing failed for item 0: TensorArray has size zero, but element shape [?,128] is not fully defined. Currently only static shapes are supported when packing zero-size TensorArrays.
	 [[Node: style_encoder/bidirectional/TensorArrayStack/TensorArrayGatherV3 = TensorArrayGatherV3[_class=["loc:@style_encoder/bidirectional/TensorArray"], dtype=DT_FLOAT, element_shape=[?,128], _device="/job:localhost/replica:0/task:0/device:CPU:0"](style_encoder/bidirectional/TensorArray, style_encoder/bidirectional/TensorArrayStack/range, style_encoder/bidirectional/while/Exit_1)]]

Caused by op 'style_encoder/bidirectional/TensorArrayStack/TensorArrayGatherV3', defined at:
  File "/usr/local/lib/python3.6/threading.py", line 884, in _bootstrap
    self._bootstrap_inner()
  File "/usr/local/lib/python3.6/threading.py", line 916, in _bootstrap_inner
    self.run()
  File "/usr/local/lib/python3.6/site-packages/anyio/_backends/_asyncio.py", line 867, in run
    result = context.run(func, *args)
  File "/usr/local/lib/python3.6/site-packages/contextvars/__init__.py", line 38, in run
    return callable(*args, **kwargs)
  File "./main.py", line 616, in convert_markdown
    initialize_model()
  File "./main.py", line 78, in initialize_model
    style_embedding_size=256
  File "./train.py", line 56, in __init__
    super(StyleSynthesisModel, self).__init__(reader=None, **kwargs)
  File "./tf_base_model.py", line 110, in __init__
    self.graph = self.build_graph()
  File "./tf_base_model.py", line 401, in build_graph
    self.loss = self.calculate_loss()
  File "./train.py", line 81, in calculate_loss
    style_emb = style_encoder(self.ref_x)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/engine/topology.py", line 258, in __call__
    output = super(Layer, self).__call__(inputs, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/layers/base.py", line 696, in __call__
    outputs = self.call(inputs, *args, **kwargs)
  File "./train.py", line 26, in call
    x = self.bi_rnn(stroke_sequence)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/engine/topology.py", line 258, in __call__
    output = super(Layer, self).__call__(inputs, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/layers/base.py", line 696, in __call__
    outputs = self.call(inputs, *args, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/layers/wrappers.py", line 346, in call
    y = self.forward_layer.call(inputs, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/layers/recurrent.py", line 2056, in call
    inputs, mask=mask, training=training, initial_state=initial_state)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/layers/recurrent.py", line 613, in call
    input_length=timesteps)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/backend.py", line 2963, in rnn
    outputs = output_ta.stack()
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/ops/tensor_array_ops.py", line 893, in stack
    return self._implementation.stack(name=name)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/ops/tensor_array_ops.py", line 291, in stack
    return self.gather(math_ops.range(0, self.size()), name=name)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/ops/tensor_array_ops.py", line 305, in gather
    element_shape=element_shape)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/ops/gen_data_flow_ops.py", line 4186, in _tensor_array_gather_v3
    flow_in=flow_in, dtype=dtype, element_shape=element_shape, name=name)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/framework/op_def_library.py", line 787, in _apply_op_helper
    op_def=op_def)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/framework/ops.py", line 3271, in create_op
    op_def=op_def)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/framework/ops.py", line 1650, in __init__
    self._traceback = self._graph._extract_stack()  # pylint: disable=protected-access

UnimplementedError (see above for traceback): TensorArray has size zero, but element shape [?,128] is not fully defined. Currently only static shapes are supported when packing zero-size TensorArrays.
	 [[Node: style_encoder/bidirectional/TensorArrayStack/TensorArrayGatherV3 = TensorArrayGatherV3[_class=["loc:@style_encoder/bidirectional/TensorArray"], dtype=DT_FLOAT, element_shape=[?,128], _device="/job:localhost/replica:0/task:0/device:CPU:0"](style_encoder/bidirectional/TensorArray, style_encoder/bidirectional/TensorArrayStack/range, style_encoder/bidirectional/while/Exit_1)]]

[[05/11/2025 09:13:41 AM]] Error during model inference: TensorArray has size zero, but element shape [?,128] is not fully defined. Currently only static shapes are supported when packing zero-size TensorArrays.
	 [[Node: style_encoder/bidirectional/TensorArrayStack/TensorArrayGatherV3 = TensorArrayGatherV3[_class=["loc:@style_encoder/bidirectional/TensorArray"], dtype=DT_FLOAT, element_shape=[?,128], _device="/job:localhost/replica:0/task:0/device:CPU:0"](style_encoder/bidirectional/TensorArray, style_encoder/bidirectional/TensorArrayStack/range, style_encoder/bidirectional/while/Exit_1)]]

Caused by op 'style_encoder/bidirectional/TensorArrayStack/TensorArrayGatherV3', defined at:
  File "/usr/local/lib/python3.6/threading.py", line 884, in _bootstrap
    self._bootstrap_inner()
  File "/usr/local/lib/python3.6/threading.py", line 916, in _bootstrap_inner
    self.run()
  File "/usr/local/lib/python3.6/site-packages/anyio/_backends/_asyncio.py", line 867, in run
    result = context.run(func, *args)
  File "/usr/local/lib/python3.6/site-packages/contextvars/__init__.py", line 38, in run
    return callable(*args, **kwargs)
  File "./main.py", line 616, in convert_markdown
    initialize_model()
  File "./main.py", line 78, in initialize_model
    style_embedding_size=256
  File "./train.py", line 56, in __init__
    super(StyleSynthesisModel, self).__init__(reader=None, **kwargs)
  File "./tf_base_model.py", line 110, in __init__
    self.graph = self.build_graph()
  File "./tf_base_model.py", line 401, in build_graph
    self.loss = self.calculate_loss()
  File "./train.py", line 81, in calculate_loss
    style_emb = style_encoder(self.ref_x)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/engine/topology.py", line 258, in __call__
    output = super(Layer, self).__call__(inputs, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/layers/base.py", line 696, in __call__
    outputs = self.call(inputs, *args, **kwargs)
  File "./train.py", line 26, in call
    x = self.bi_rnn(stroke_sequence)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/engine/topology.py", line 258, in __call__
    output = super(Layer, self).__call__(inputs, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/layers/base.py", line 696, in __call__
    outputs = self.call(inputs, *args, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/layers/wrappers.py", line 346, in call
    y = self.forward_layer.call(inputs, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/layers/recurrent.py", line 2056, in call
    inputs, mask=mask, training=training, initial_state=initial_state)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/layers/recurrent.py", line 613, in call
    input_length=timesteps)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/backend.py", line 2963, in rnn
    outputs = output_ta.stack()
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/ops/tensor_array_ops.py", line 893, in stack
    return self._implementation.stack(name=name)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/ops/tensor_array_ops.py", line 291, in stack
    return self.gather(math_ops.range(0, self.size()), name=name)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/ops/tensor_array_ops.py", line 305, in gather
    element_shape=element_shape)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/ops/gen_data_flow_ops.py", line 4186, in _tensor_array_gather_v3
    flow_in=flow_in, dtype=dtype, element_shape=element_shape, name=name)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/framework/op_def_library.py", line 787, in _apply_op_helper
    op_def=op_def)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/framework/ops.py", line 3271, in create_op
    op_def=op_def)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/framework/ops.py", line 1650, in __init__
    self._traceback = self._graph._extract_stack()  # pylint: disable=protected-access

UnimplementedError (see above for traceback): TensorArray has size zero, but element shape [?,128] is not fully defined. Currently only static shapes are supported when packing zero-size TensorArrays.
	 [[Node: style_encoder/bidirectional/TensorArrayStack/TensorArrayGatherV3 = TensorArrayGatherV3[_class=["loc:@style_encoder/bidirectional/TensorArray"], dtype=DT_FLOAT, element_shape=[?,128], _device="/job:localhost/replica:0/task:0/device:CPU:0"](style_encoder/bidirectional/TensorArray, style_encoder/bidirectional/TensorArrayStack/range, style_encoder/bidirectional/while/Exit_1)]]

[[05/11/2025 09:13:41 AM]] Processing failed for item 2: TensorArray has size zero, but element shape [?,128] is not fully defined. Currently only static shapes are supported when packing zero-size TensorArrays.
	 [[Node: style_encoder/bidirectional/TensorArrayStack/TensorArrayGatherV3 = TensorArrayGatherV3[_class=["loc:@style_encoder/bidirectional/TensorArray"], dtype=DT_FLOAT, element_shape=[?,128], _device="/job:localhost/replica:0/task:0/device:CPU:0"](style_encoder/bidirectional/TensorArray, style_encoder/bidirectional/TensorArrayStack/range, style_encoder/bidirectional/while/Exit_1)]]

Caused by op 'style_encoder/bidirectional/TensorArrayStack/TensorArrayGatherV3', defined at:
  File "/usr/local/lib/python3.6/threading.py", line 884, in _bootstrap
    self._bootstrap_inner()
  File "/usr/local/lib/python3.6/threading.py", line 916, in _bootstrap_inner
    self.run()
  File "/usr/local/lib/python3.6/site-packages/anyio/_backends/_asyncio.py", line 867, in run
    result = context.run(func, *args)
  File "/usr/local/lib/python3.6/site-packages/contextvars/__init__.py", line 38, in run
    return callable(*args, **kwargs)
  File "./main.py", line 616, in convert_markdown
    initialize_model()
  File "./main.py", line 78, in initialize_model
    style_embedding_size=256
  File "./train.py", line 56, in __init__
    super(StyleSynthesisModel, self).__init__(reader=None, **kwargs)
  File "./tf_base_model.py", line 110, in __init__
    self.graph = self.build_graph()
  File "./tf_base_model.py", line 401, in build_graph
    self.loss = self.calculate_loss()
  File "./train.py", line 81, in calculate_loss
    style_emb = style_encoder(self.ref_x)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/engine/topology.py", line 258, in __call__
    output = super(Layer, self).__call__(inputs, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/layers/base.py", line 696, in __call__
    outputs = self.call(inputs, *args, **kwargs)
  File "./train.py", line 26, in call
    x = self.bi_rnn(stroke_sequence)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/engine/topology.py", line 258, in __call__
    output = super(Layer, self).__call__(inputs, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/layers/base.py", line 696, in __call__
    outputs = self.call(inputs, *args, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/layers/wrappers.py", line 346, in call
    y = self.forward_layer.call(inputs, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/layers/recurrent.py", line 2056, in call
    inputs, mask=mask, training=training, initial_state=initial_state)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/layers/recurrent.py", line 613, in call
    input_length=timesteps)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/backend.py", line 2963, in rnn
    outputs = output_ta.stack()
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/ops/tensor_array_ops.py", line 893, in stack
    return self._implementation.stack(name=name)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/ops/tensor_array_ops.py", line 291, in stack
    return self.gather(math_ops.range(0, self.size()), name=name)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/ops/tensor_array_ops.py", line 305, in gather
    element_shape=element_shape)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/ops/gen_data_flow_ops.py", line 4186, in _tensor_array_gather_v3
    flow_in=flow_in, dtype=dtype, element_shape=element_shape, name=name)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/framework/op_def_library.py", line 787, in _apply_op_helper
    op_def=op_def)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/framework/ops.py", line 3271, in create_op
    op_def=op_def)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/framework/ops.py", line 1650, in __init__
    self._traceback = self._graph._extract_stack()  # pylint: disable=protected-access

UnimplementedError (see above for traceback): TensorArray has size zero, but element shape [?,128] is not fully defined. Currently only static shapes are supported when packing zero-size TensorArrays.
	 [[Node: style_encoder/bidirectional/TensorArrayStack/TensorArrayGatherV3 = TensorArrayGatherV3[_class=["loc:@style_encoder/bidirectional/TensorArray"], dtype=DT_FLOAT, element_shape=[?,128], _device="/job:localhost/replica:0/task:0/device:CPU:0"](style_encoder/bidirectional/TensorArray, style_encoder/bidirectional/TensorArrayStack/range, style_encoder/bidirectional/while/Exit_1)]]

[[05/11/2025 09:13:41 AM]] Error during model inference: TensorArray has size zero, but element shape [?,128] is not fully defined. Currently only static shapes are supported when packing zero-size TensorArrays.
	 [[Node: style_encoder/bidirectional/TensorArrayStack_1/TensorArrayGatherV3 = TensorArrayGatherV3[_class=["loc:@style_encoder/bidirectional/TensorArray_2"], dtype=DT_FLOAT, element_shape=[?,128], _device="/job:localhost/replica:0/task:0/device:CPU:0"](style_encoder/bidirectional/TensorArray_2, style_encoder/bidirectional/TensorArrayStack_1/range, style_encoder/bidirectional/while_1/Exit_1)]]

Caused by op 'style_encoder/bidirectional/TensorArrayStack_1/TensorArrayGatherV3', defined at:
  File "/usr/local/lib/python3.6/threading.py", line 884, in _bootstrap
    self._bootstrap_inner()
  File "/usr/local/lib/python3.6/threading.py", line 916, in _bootstrap_inner
    self.run()
  File "/usr/local/lib/python3.6/site-packages/anyio/_backends/_asyncio.py", line 867, in run
    result = context.run(func, *args)
  File "/usr/local/lib/python3.6/site-packages/contextvars/__init__.py", line 38, in run
    return callable(*args, **kwargs)
  File "./main.py", line 616, in convert_markdown
    initialize_model()
  File "./main.py", line 78, in initialize_model
    style_embedding_size=256
  File "./train.py", line 56, in __init__
    super(StyleSynthesisModel, self).__init__(reader=None, **kwargs)
  File "./tf_base_model.py", line 110, in __init__
    self.graph = self.build_graph()
  File "./tf_base_model.py", line 401, in build_graph
    self.loss = self.calculate_loss()
  File "./train.py", line 81, in calculate_loss
    style_emb = style_encoder(self.ref_x)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/engine/topology.py", line 258, in __call__
    output = super(Layer, self).__call__(inputs, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/layers/base.py", line 696, in __call__
    outputs = self.call(inputs, *args, **kwargs)
  File "./train.py", line 26, in call
    x = self.bi_rnn(stroke_sequence)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/engine/topology.py", line 258, in __call__
    output = super(Layer, self).__call__(inputs, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/layers/base.py", line 696, in __call__
    outputs = self.call(inputs, *args, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/layers/wrappers.py", line 347, in call
    y_rev = self.backward_layer.call(inputs, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/layers/recurrent.py", line 2056, in call
    inputs, mask=mask, training=training, initial_state=initial_state)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/layers/recurrent.py", line 613, in call
    input_length=timesteps)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/backend.py", line 2963, in rnn
    outputs = output_ta.stack()
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/ops/tensor_array_ops.py", line 893, in stack
    return self._implementation.stack(name=name)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/ops/tensor_array_ops.py", line 291, in stack
    return self.gather(math_ops.range(0, self.size()), name=name)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/ops/tensor_array_ops.py", line 305, in gather
    element_shape=element_shape)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/ops/gen_data_flow_ops.py", line 4186, in _tensor_array_gather_v3
    flow_in=flow_in, dtype=dtype, element_shape=element_shape, name=name)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/framework/op_def_library.py", line 787, in _apply_op_helper
    op_def=op_def)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/framework/ops.py", line 3271, in create_op
    op_def=op_def)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/framework/ops.py", line 1650, in __init__
    self._traceback = self._graph._extract_stack()  # pylint: disable=protected-access

UnimplementedError (see above for traceback): TensorArray has size zero, but element shape [?,128] is not fully defined. Currently only static shapes are supported when packing zero-size TensorArrays.
	 [[Node: style_encoder/bidirectional/TensorArrayStack_1/TensorArrayGatherV3 = TensorArrayGatherV3[_class=["loc:@style_encoder/bidirectional/TensorArray_2"], dtype=DT_FLOAT, element_shape=[?,128], _device="/job:localhost/replica:0/task:0/device:CPU:0"](style_encoder/bidirectional/TensorArray_2, style_encoder/bidirectional/TensorArrayStack_1/range, style_encoder/bidirectional/while_1/Exit_1)]]

[[05/11/2025 09:13:41 AM]] Processing failed for item 3: TensorArray has size zero, but element shape [?,128] is not fully defined. Currently only static shapes are supported when packing zero-size TensorArrays.
	 [[Node: style_encoder/bidirectional/TensorArrayStack_1/TensorArrayGatherV3 = TensorArrayGatherV3[_class=["loc:@style_encoder/bidirectional/TensorArray_2"], dtype=DT_FLOAT, element_shape=[?,128], _device="/job:localhost/replica:0/task:0/device:CPU:0"](style_encoder/bidirectional/TensorArray_2, style_encoder/bidirectional/TensorArrayStack_1/range, style_encoder/bidirectional/while_1/Exit_1)]]

Caused by op 'style_encoder/bidirectional/TensorArrayStack_1/TensorArrayGatherV3', defined at:
  File "/usr/local/lib/python3.6/threading.py", line 884, in _bootstrap
    self._bootstrap_inner()
  File "/usr/local/lib/python3.6/threading.py", line 916, in _bootstrap_inner
    self.run()
  File "/usr/local/lib/python3.6/site-packages/anyio/_backends/_asyncio.py", line 867, in run
    result = context.run(func, *args)
  File "/usr/local/lib/python3.6/site-packages/contextvars/__init__.py", line 38, in run
    return callable(*args, **kwargs)
  File "./main.py", line 616, in convert_markdown
    initialize_model()
  File "./main.py", line 78, in initialize_model
    style_embedding_size=256
  File "./train.py", line 56, in __init__
    super(StyleSynthesisModel, self).__init__(reader=None, **kwargs)
  File "./tf_base_model.py", line 110, in __init__
    self.graph = self.build_graph()
  File "./tf_base_model.py", line 401, in build_graph
    self.loss = self.calculate_loss()
  File "./train.py", line 81, in calculate_loss
    style_emb = style_encoder(self.ref_x)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/engine/topology.py", line 258, in __call__
    output = super(Layer, self).__call__(inputs, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/layers/base.py", line 696, in __call__
    outputs = self.call(inputs, *args, **kwargs)
  File "./train.py", line 26, in call
    x = self.bi_rnn(stroke_sequence)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/engine/topology.py", line 258, in __call__
    output = super(Layer, self).__call__(inputs, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/layers/base.py", line 696, in __call__
    outputs = self.call(inputs, *args, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/layers/wrappers.py", line 347, in call
    y_rev = self.backward_layer.call(inputs, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/layers/recurrent.py", line 2056, in call
    inputs, mask=mask, training=training, initial_state=initial_state)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/layers/recurrent.py", line 613, in call
    input_length=timesteps)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/backend.py", line 2963, in rnn
    outputs = output_ta.stack()
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/ops/tensor_array_ops.py", line 893, in stack
    return self._implementation.stack(name=name)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/ops/tensor_array_ops.py", line 291, in stack
    return self.gather(math_ops.range(0, self.size()), name=name)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/ops/tensor_array_ops.py", line 305, in gather
    element_shape=element_shape)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/ops/gen_data_flow_ops.py", line 4186, in _tensor_array_gather_v3
    flow_in=flow_in, dtype=dtype, element_shape=element_shape, name=name)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/framework/op_def_library.py", line 787, in _apply_op_helper
    op_def=op_def)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/framework/ops.py", line 3271, in create_op
    op_def=op_def)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/framework/ops.py", line 1650, in __init__
    self._traceback = self._graph._extract_stack()  # pylint: disable=protected-access

UnimplementedError (see above for traceback): TensorArray has size zero, but element shape [?,128] is not fully defined. Currently only static shapes are supported when packing zero-size TensorArrays.
	 [[Node: style_encoder/bidirectional/TensorArrayStack_1/TensorArrayGatherV3 = TensorArrayGatherV3[_class=["loc:@style_encoder/bidirectional/TensorArray_2"], dtype=DT_FLOAT, element_shape=[?,128], _device="/job:localhost/replica:0/task:0/device:CPU:0"](style_encoder/bidirectional/TensorArray_2, style_encoder/bidirectional/TensorArrayStack_1/range, style_encoder/bidirectional/while_1/Exit_1)]]

[[05/11/2025 09:13:41 AM]] Error during model inference: TensorArray has size zero, but element shape [?,128] is not fully defined. Currently only static shapes are supported when packing zero-size TensorArrays.
	 [[Node: style_encoder/bidirectional/TensorArrayStack/TensorArrayGatherV3 = TensorArrayGatherV3[_class=["loc:@style_encoder/bidirectional/TensorArray"], dtype=DT_FLOAT, element_shape=[?,128], _device="/job:localhost/replica:0/task:0/device:CPU:0"](style_encoder/bidirectional/TensorArray, style_encoder/bidirectional/TensorArrayStack/range, style_encoder/bidirectional/while/Exit_1)]]

Caused by op 'style_encoder/bidirectional/TensorArrayStack/TensorArrayGatherV3', defined at:
  File "/usr/local/lib/python3.6/threading.py", line 884, in _bootstrap
    self._bootstrap_inner()
  File "/usr/local/lib/python3.6/threading.py", line 916, in _bootstrap_inner
    self.run()
  File "/usr/local/lib/python3.6/site-packages/anyio/_backends/_asyncio.py", line 867, in run
    result = context.run(func, *args)
  File "/usr/local/lib/python3.6/site-packages/contextvars/__init__.py", line 38, in run
    return callable(*args, **kwargs)
  File "./main.py", line 616, in convert_markdown
    initialize_model()
  File "./main.py", line 78, in initialize_model
    style_embedding_size=256
  File "./train.py", line 56, in __init__
    super(StyleSynthesisModel, self).__init__(reader=None, **kwargs)
  File "./tf_base_model.py", line 110, in __init__
    self.graph = self.build_graph()
  File "./tf_base_model.py", line 401, in build_graph
    self.loss = self.calculate_loss()
  File "./train.py", line 81, in calculate_loss
    style_emb = style_encoder(self.ref_x)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/engine/topology.py", line 258, in __call__
    output = super(Layer, self).__call__(inputs, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/layers/base.py", line 696, in __call__
    outputs = self.call(inputs, *args, **kwargs)
  File "./train.py", line 26, in call
    x = self.bi_rnn(stroke_sequence)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/engine/topology.py", line 258, in __call__
    output = super(Layer, self).__call__(inputs, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/layers/base.py", line 696, in __call__
    outputs = self.call(inputs, *args, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/layers/wrappers.py", line 346, in call
    y = self.forward_layer.call(inputs, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/layers/recurrent.py", line 2056, in call
    inputs, mask=mask, training=training, initial_state=initial_state)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/layers/recurrent.py", line 613, in call
    input_length=timesteps)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/backend.py", line 2963, in rnn
    outputs = output_ta.stack()
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/ops/tensor_array_ops.py", line 893, in stack
    return self._implementation.stack(name=name)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/ops/tensor_array_ops.py", line 291, in stack
    return self.gather(math_ops.range(0, self.size()), name=name)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/ops/tensor_array_ops.py", line 305, in gather
    element_shape=element_shape)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/ops/gen_data_flow_ops.py", line 4186, in _tensor_array_gather_v3
    flow_in=flow_in, dtype=dtype, element_shape=element_shape, name=name)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/framework/op_def_library.py", line 787, in _apply_op_helper
    op_def=op_def)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/framework/ops.py", line 3271, in create_op
    op_def=op_def)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/framework/ops.py", line 1650, in __init__
    self._traceback = self._graph._extract_stack()  # pylint: disable=protected-access

UnimplementedError (see above for traceback): TensorArray has size zero, but element shape [?,128] is not fully defined. Currently only static shapes are supported when packing zero-size TensorArrays.
	 [[Node: style_encoder/bidirectional/TensorArrayStack/TensorArrayGatherV3 = TensorArrayGatherV3[_class=["loc:@style_encoder/bidirectional/TensorArray"], dtype=DT_FLOAT, element_shape=[?,128], _device="/job:localhost/replica:0/task:0/device:CPU:0"](style_encoder/bidirectional/TensorArray, style_encoder/bidirectional/TensorArrayStack/range, style_encoder/bidirectional/while/Exit_1)]]

[[05/11/2025 09:13:41 AM]] Processing failed for item 4: TensorArray has size zero, but element shape [?,128] is not fully defined. Currently only static shapes are supported when packing zero-size TensorArrays.
	 [[Node: style_encoder/bidirectional/TensorArrayStack/TensorArrayGatherV3 = TensorArrayGatherV3[_class=["loc:@style_encoder/bidirectional/TensorArray"], dtype=DT_FLOAT, element_shape=[?,128], _device="/job:localhost/replica:0/task:0/device:CPU:0"](style_encoder/bidirectional/TensorArray, style_encoder/bidirectional/TensorArrayStack/range, style_encoder/bidirectional/while/Exit_1)]]

Caused by op 'style_encoder/bidirectional/TensorArrayStack/TensorArrayGatherV3', defined at:
  File "/usr/local/lib/python3.6/threading.py", line 884, in _bootstrap
    self._bootstrap_inner()
  File "/usr/local/lib/python3.6/threading.py", line 916, in _bootstrap_inner
    self.run()
  File "/usr/local/lib/python3.6/site-packages/anyio/_backends/_asyncio.py", line 867, in run
    result = context.run(func, *args)
  File "/usr/local/lib/python3.6/site-packages/contextvars/__init__.py", line 38, in run
    return callable(*args, **kwargs)
  File "./main.py", line 616, in convert_markdown
    initialize_model()
  File "./main.py", line 78, in initialize_model
    style_embedding_size=256
  File "./train.py", line 56, in __init__
    super(StyleSynthesisModel, self).__init__(reader=None, **kwargs)
  File "./tf_base_model.py", line 110, in __init__
    self.graph = self.build_graph()
  File "./tf_base_model.py", line 401, in build_graph
    self.loss = self.calculate_loss()
  File "./train.py", line 81, in calculate_loss
    style_emb = style_encoder(self.ref_x)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/engine/topology.py", line 258, in __call__
    output = super(Layer, self).__call__(inputs, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/layers/base.py", line 696, in __call__
    outputs = self.call(inputs, *args, **kwargs)
  File "./train.py", line 26, in call
    x = self.bi_rnn(stroke_sequence)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/engine/topology.py", line 258, in __call__
    output = super(Layer, self).__call__(inputs, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/layers/base.py", line 696, in __call__
    outputs = self.call(inputs, *args, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/layers/wrappers.py", line 346, in call
    y = self.forward_layer.call(inputs, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/layers/recurrent.py", line 2056, in call
    inputs, mask=mask, training=training, initial_state=initial_state)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/layers/recurrent.py", line 613, in call
    input_length=timesteps)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/backend.py", line 2963, in rnn
    outputs = output_ta.stack()
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/ops/tensor_array_ops.py", line 893, in stack
    return self._implementation.stack(name=name)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/ops/tensor_array_ops.py", line 291, in stack
    return self.gather(math_ops.range(0, self.size()), name=name)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/ops/tensor_array_ops.py", line 305, in gather
    element_shape=element_shape)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/ops/gen_data_flow_ops.py", line 4186, in _tensor_array_gather_v3
    flow_in=flow_in, dtype=dtype, element_shape=element_shape, name=name)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/framework/op_def_library.py", line 787, in _apply_op_helper
    op_def=op_def)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/framework/ops.py", line 3271, in create_op
    op_def=op_def)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/framework/ops.py", line 1650, in __init__
    self._traceback = self._graph._extract_stack()  # pylint: disable=protected-access

UnimplementedError (see above for traceback): TensorArray has size zero, but element shape [?,128] is not fully defined. Currently only static shapes are supported when packing zero-size TensorArrays.
	 [[Node: style_encoder/bidirectional/TensorArrayStack/TensorArrayGatherV3 = TensorArrayGatherV3[_class=["loc:@style_encoder/bidirectional/TensorArray"], dtype=DT_FLOAT, element_shape=[?,128], _device="/job:localhost/replica:0/task:0/device:CPU:0"](style_encoder/bidirectional/TensorArray, style_encoder/bidirectional/TensorArrayStack/range, style_encoder/bidirectional/while/Exit_1)]]

[[05/11/2025 09:13:41 AM]] Error during model inference: TensorArray has size zero, but element shape [?,128] is not fully defined. Currently only static shapes are supported when packing zero-size TensorArrays.
	 [[Node: style_encoder/bidirectional/TensorArrayStack/TensorArrayGatherV3 = TensorArrayGatherV3[_class=["loc:@style_encoder/bidirectional/TensorArray"], dtype=DT_FLOAT, element_shape=[?,128], _device="/job:localhost/replica:0/task:0/device:CPU:0"](style_encoder/bidirectional/TensorArray, style_encoder/bidirectional/TensorArrayStack/range, style_encoder/bidirectional/while/Exit_1)]]

Caused by op 'style_encoder/bidirectional/TensorArrayStack/TensorArrayGatherV3', defined at:
  File "/usr/local/lib/python3.6/threading.py", line 884, in _bootstrap
    self._bootstrap_inner()
  File "/usr/local/lib/python3.6/threading.py", line 916, in _bootstrap_inner
    self.run()
  File "/usr/local/lib/python3.6/site-packages/anyio/_backends/_asyncio.py", line 867, in run
    result = context.run(func, *args)
  File "/usr/local/lib/python3.6/site-packages/contextvars/__init__.py", line 38, in run
    return callable(*args, **kwargs)
  File "./main.py", line 616, in convert_markdown
    initialize_model()
  File "./main.py", line 78, in initialize_model
    style_embedding_size=256
  File "./train.py", line 56, in __init__
    super(StyleSynthesisModel, self).__init__(reader=None, **kwargs)
  File "./tf_base_model.py", line 110, in __init__
    self.graph = self.build_graph()
  File "./tf_base_model.py", line 401, in build_graph
    self.loss = self.calculate_loss()
  File "./train.py", line 81, in calculate_loss
    style_emb = style_encoder(self.ref_x)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/engine/topology.py", line 258, in __call__
    output = super(Layer, self).__call__(inputs, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/layers/base.py", line 696, in __call__
    outputs = self.call(inputs, *args, **kwargs)
  File "./train.py", line 26, in call
    x = self.bi_rnn(stroke_sequence)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/engine/topology.py", line 258, in __call__
    output = super(Layer, self).__call__(inputs, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/layers/base.py", line 696, in __call__
    outputs = self.call(inputs, *args, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/layers/wrappers.py", line 346, in call
    y = self.forward_layer.call(inputs, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/layers/recurrent.py", line 2056, in call
    inputs, mask=mask, training=training, initial_state=initial_state)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/layers/recurrent.py", line 613, in call
    input_length=timesteps)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/backend.py", line 2963, in rnn
    outputs = output_ta.stack()
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/ops/tensor_array_ops.py", line 893, in stack
    return self._implementation.stack(name=name)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/ops/tensor_array_ops.py", line 291, in stack
    return self.gather(math_ops.range(0, self.size()), name=name)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/ops/tensor_array_ops.py", line 305, in gather
    element_shape=element_shape)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/ops/gen_data_flow_ops.py", line 4186, in _tensor_array_gather_v3
    flow_in=flow_in, dtype=dtype, element_shape=element_shape, name=name)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/framework/op_def_library.py", line 787, in _apply_op_helper
    op_def=op_def)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/framework/ops.py", line 3271, in create_op
    op_def=op_def)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/framework/ops.py", line 1650, in __init__
    self._traceback = self._graph._extract_stack()  # pylint: disable=protected-access

UnimplementedError (see above for traceback): TensorArray has size zero, but element shape [?,128] is not fully defined. Currently only static shapes are supported when packing zero-size TensorArrays.
	 [[Node: style_encoder/bidirectional/TensorArrayStack/TensorArrayGatherV3 = TensorArrayGatherV3[_class=["loc:@style_encoder/bidirectional/TensorArray"], dtype=DT_FLOAT, element_shape=[?,128], _device="/job:localhost/replica:0/task:0/device:CPU:0"](style_encoder/bidirectional/TensorArray, style_encoder/bidirectional/TensorArrayStack/range, style_encoder/bidirectional/while/Exit_1)]]

[[05/11/2025 09:13:41 AM]] Processing failed for item 6: TensorArray has size zero, but element shape [?,128] is not fully defined. Currently only static shapes are supported when packing zero-size TensorArrays.
	 [[Node: style_encoder/bidirectional/TensorArrayStack/TensorArrayGatherV3 = TensorArrayGatherV3[_class=["loc:@style_encoder/bidirectional/TensorArray"], dtype=DT_FLOAT, element_shape=[?,128], _device="/job:localhost/replica:0/task:0/device:CPU:0"](style_encoder/bidirectional/TensorArray, style_encoder/bidirectional/TensorArrayStack/range, style_encoder/bidirectional/while/Exit_1)]]

Caused by op 'style_encoder/bidirectional/TensorArrayStack/TensorArrayGatherV3', defined at:
  File "/usr/local/lib/python3.6/threading.py", line 884, in _bootstrap
    self._bootstrap_inner()
  File "/usr/local/lib/python3.6/threading.py", line 916, in _bootstrap_inner
    self.run()
  File "/usr/local/lib/python3.6/site-packages/anyio/_backends/_asyncio.py", line 867, in run
    result = context.run(func, *args)
  File "/usr/local/lib/python3.6/site-packages/contextvars/__init__.py", line 38, in run
    return callable(*args, **kwargs)
  File "./main.py", line 616, in convert_markdown
    initialize_model()
  File "./main.py", line 78, in initialize_model
    style_embedding_size=256
  File "./train.py", line 56, in __init__
    super(StyleSynthesisModel, self).__init__(reader=None, **kwargs)
  File "./tf_base_model.py", line 110, in __init__
    self.graph = self.build_graph()
  File "./tf_base_model.py", line 401, in build_graph
    self.loss = self.calculate_loss()
  File "./train.py", line 81, in calculate_loss
    style_emb = style_encoder(self.ref_x)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/engine/topology.py", line 258, in __call__
    output = super(Layer, self).__call__(inputs, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/layers/base.py", line 696, in __call__
    outputs = self.call(inputs, *args, **kwargs)
  File "./train.py", line 26, in call
    x = self.bi_rnn(stroke_sequence)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/engine/topology.py", line 258, in __call__
    output = super(Layer, self).__call__(inputs, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/layers/base.py", line 696, in __call__
    outputs = self.call(inputs, *args, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/layers/wrappers.py", line 346, in call
    y = self.forward_layer.call(inputs, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/layers/recurrent.py", line 2056, in call
    inputs, mask=mask, training=training, initial_state=initial_state)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/layers/recurrent.py", line 613, in call
    input_length=timesteps)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/backend.py", line 2963, in rnn
    outputs = output_ta.stack()
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/ops/tensor_array_ops.py", line 893, in stack
    return self._implementation.stack(name=name)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/ops/tensor_array_ops.py", line 291, in stack
    return self.gather(math_ops.range(0, self.size()), name=name)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/ops/tensor_array_ops.py", line 305, in gather
    element_shape=element_shape)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/ops/gen_data_flow_ops.py", line 4186, in _tensor_array_gather_v3
    flow_in=flow_in, dtype=dtype, element_shape=element_shape, name=name)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/framework/op_def_library.py", line 787, in _apply_op_helper
    op_def=op_def)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/framework/ops.py", line 3271, in create_op
    op_def=op_def)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/framework/ops.py", line 1650, in __init__
    self._traceback = self._graph._extract_stack()  # pylint: disable=protected-access

UnimplementedError (see above for traceback): TensorArray has size zero, but element shape [?,128] is not fully defined. Currently only static shapes are supported when packing zero-size TensorArrays.
	 [[Node: style_encoder/bidirectional/TensorArrayStack/TensorArrayGatherV3 = TensorArrayGatherV3[_class=["loc:@style_encoder/bidirectional/TensorArray"], dtype=DT_FLOAT, element_shape=[?,128], _device="/job:localhost/replica:0/task:0/device:CPU:0"](style_encoder/bidirectional/TensorArray, style_encoder/bidirectional/TensorArrayStack/range, style_encoder/bidirectional/while/Exit_1)]]

[[05/11/2025 09:13:41 AM]] Error during model inference: TensorArray has size zero, but element shape [?,128] is not fully defined. Currently only static shapes are supported when packing zero-size TensorArrays.
	 [[Node: style_encoder/bidirectional/TensorArrayStack_1/TensorArrayGatherV3 = TensorArrayGatherV3[_class=["loc:@style_encoder/bidirectional/TensorArray_2"], dtype=DT_FLOAT, element_shape=[?,128], _device="/job:localhost/replica:0/task:0/device:CPU:0"](style_encoder/bidirectional/TensorArray_2, style_encoder/bidirectional/TensorArrayStack_1/range, style_encoder/bidirectional/while_1/Exit_1)]]

Caused by op 'style_encoder/bidirectional/TensorArrayStack_1/TensorArrayGatherV3', defined at:
  File "/usr/local/lib/python3.6/threading.py", line 884, in _bootstrap
    self._bootstrap_inner()
  File "/usr/local/lib/python3.6/threading.py", line 916, in _bootstrap_inner
    self.run()
  File "/usr/local/lib/python3.6/site-packages/anyio/_backends/_asyncio.py", line 867, in run
    result = context.run(func, *args)
  File "/usr/local/lib/python3.6/site-packages/contextvars/__init__.py", line 38, in run
    return callable(*args, **kwargs)
  File "./main.py", line 616, in convert_markdown
    initialize_model()
  File "./main.py", line 78, in initialize_model
    style_embedding_size=256
  File "./train.py", line 56, in __init__
    super(StyleSynthesisModel, self).__init__(reader=None, **kwargs)
  File "./tf_base_model.py", line 110, in __init__
    self.graph = self.build_graph()
  File "./tf_base_model.py", line 401, in build_graph
    self.loss = self.calculate_loss()
  File "./train.py", line 81, in calculate_loss
    style_emb = style_encoder(self.ref_x)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/engine/topology.py", line 258, in __call__
    output = super(Layer, self).__call__(inputs, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/layers/base.py", line 696, in __call__
    outputs = self.call(inputs, *args, **kwargs)
  File "./train.py", line 26, in call
    x = self.bi_rnn(stroke_sequence)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/engine/topology.py", line 258, in __call__
    output = super(Layer, self).__call__(inputs, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/layers/base.py", line 696, in __call__
    outputs = self.call(inputs, *args, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/layers/wrappers.py", line 347, in call
    y_rev = self.backward_layer.call(inputs, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/layers/recurrent.py", line 2056, in call
    inputs, mask=mask, training=training, initial_state=initial_state)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/layers/recurrent.py", line 613, in call
    input_length=timesteps)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/backend.py", line 2963, in rnn
    outputs = output_ta.stack()
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/ops/tensor_array_ops.py", line 893, in stack
    return self._implementation.stack(name=name)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/ops/tensor_array_ops.py", line 291, in stack
    return self.gather(math_ops.range(0, self.size()), name=name)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/ops/tensor_array_ops.py", line 305, in gather
    element_shape=element_shape)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/ops/gen_data_flow_ops.py", line 4186, in _tensor_array_gather_v3
    flow_in=flow_in, dtype=dtype, element_shape=element_shape, name=name)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/framework/op_def_library.py", line 787, in _apply_op_helper
    op_def=op_def)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/framework/ops.py", line 3271, in create_op
    op_def=op_def)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/framework/ops.py", line 1650, in __init__
    self._traceback = self._graph._extract_stack()  # pylint: disable=protected-access

UnimplementedError (see above for traceback): TensorArray has size zero, but element shape [?,128] is not fully defined. Currently only static shapes are supported when packing zero-size TensorArrays.
	 [[Node: style_encoder/bidirectional/TensorArrayStack_1/TensorArrayGatherV3 = TensorArrayGatherV3[_class=["loc:@style_encoder/bidirectional/TensorArray_2"], dtype=DT_FLOAT, element_shape=[?,128], _device="/job:localhost/replica:0/task:0/device:CPU:0"](style_encoder/bidirectional/TensorArray_2, style_encoder/bidirectional/TensorArrayStack_1/range, style_encoder/bidirectional/while_1/Exit_1)]]

[[05/11/2025 09:13:41 AM]] Processing failed for item 5: TensorArray has size zero, but element shape [?,128] is not fully defined. Currently only static shapes are supported when packing zero-size TensorArrays.
	 [[Node: style_encoder/bidirectional/TensorArrayStack_1/TensorArrayGatherV3 = TensorArrayGatherV3[_class=["loc:@style_encoder/bidirectional/TensorArray_2"], dtype=DT_FLOAT, element_shape=[?,128], _device="/job:localhost/replica:0/task:0/device:CPU:0"](style_encoder/bidirectional/TensorArray_2, style_encoder/bidirectional/TensorArrayStack_1/range, style_encoder/bidirectional/while_1/Exit_1)]]

Caused by op 'style_encoder/bidirectional/TensorArrayStack_1/TensorArrayGatherV3', defined at:
  File "/usr/local/lib/python3.6/threading.py", line 884, in _bootstrap
    self._bootstrap_inner()
  File "/usr/local/lib/python3.6/threading.py", line 916, in _bootstrap_inner
    self.run()
  File "/usr/local/lib/python3.6/site-packages/anyio/_backends/_asyncio.py", line 867, in run
    result = context.run(func, *args)
  File "/usr/local/lib/python3.6/site-packages/contextvars/__init__.py", line 38, in run
    return callable(*args, **kwargs)
  File "./main.py", line 616, in convert_markdown
    initialize_model()
  File "./main.py", line 78, in initialize_model
    style_embedding_size=256
  File "./train.py", line 56, in __init__
    super(StyleSynthesisModel, self).__init__(reader=None, **kwargs)
  File "./tf_base_model.py", line 110, in __init__
    self.graph = self.build_graph()
  File "./tf_base_model.py", line 401, in build_graph
    self.loss = self.calculate_loss()
  File "./train.py", line 81, in calculate_loss
    style_emb = style_encoder(self.ref_x)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/engine/topology.py", line 258, in __call__
    output = super(Layer, self).__call__(inputs, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/layers/base.py", line 696, in __call__
    outputs = self.call(inputs, *args, **kwargs)
  File "./train.py", line 26, in call
    x = self.bi_rnn(stroke_sequence)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/engine/topology.py", line 258, in __call__
    output = super(Layer, self).__call__(inputs, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/layers/base.py", line 696, in __call__
    outputs = self.call(inputs, *args, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/layers/wrappers.py", line 347, in call
    y_rev = self.backward_layer.call(inputs, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/layers/recurrent.py", line 2056, in call
    inputs, mask=mask, training=training, initial_state=initial_state)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/layers/recurrent.py", line 613, in call
    input_length=timesteps)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/backend.py", line 2963, in rnn
    outputs = output_ta.stack()
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/ops/tensor_array_ops.py", line 893, in stack
    return self._implementation.stack(name=name)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/ops/tensor_array_ops.py", line 291, in stack
    return self.gather(math_ops.range(0, self.size()), name=name)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/ops/tensor_array_ops.py", line 305, in gather
    element_shape=element_shape)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/ops/gen_data_flow_ops.py", line 4186, in _tensor_array_gather_v3
    flow_in=flow_in, dtype=dtype, element_shape=element_shape, name=name)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/framework/op_def_library.py", line 787, in _apply_op_helper
    op_def=op_def)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/framework/ops.py", line 3271, in create_op
    op_def=op_def)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/framework/ops.py", line 1650, in __init__
    self._traceback = self._graph._extract_stack()  # pylint: disable=protected-access

UnimplementedError (see above for traceback): TensorArray has size zero, but element shape [?,128] is not fully defined. Currently only static shapes are supported when packing zero-size TensorArrays.
	 [[Node: style_encoder/bidirectional/TensorArrayStack_1/TensorArrayGatherV3 = TensorArrayGatherV3[_class=["loc:@style_encoder/bidirectional/TensorArray_2"], dtype=DT_FLOAT, element_shape=[?,128], _device="/job:localhost/replica:0/task:0/device:CPU:0"](style_encoder/bidirectional/TensorArray_2, style_encoder/bidirectional/TensorArrayStack_1/range, style_encoder/bidirectional/while_1/Exit_1)]]

[[05/11/2025 09:13:41 AM]] Error during model inference: TensorArray has size zero, but element shape [?,128] is not fully defined. Currently only static shapes are supported when packing zero-size TensorArrays.
	 [[Node: style_encoder/bidirectional/TensorArrayStack_1/TensorArrayGatherV3 = TensorArrayGatherV3[_class=["loc:@style_encoder/bidirectional/TensorArray_2"], dtype=DT_FLOAT, element_shape=[?,128], _device="/job:localhost/replica:0/task:0/device:CPU:0"](style_encoder/bidirectional/TensorArray_2, style_encoder/bidirectional/TensorArrayStack_1/range, style_encoder/bidirectional/while_1/Exit_1)]]

Caused by op 'style_encoder/bidirectional/TensorArrayStack_1/TensorArrayGatherV3', defined at:
  File "/usr/local/lib/python3.6/threading.py", line 884, in _bootstrap
    self._bootstrap_inner()
  File "/usr/local/lib/python3.6/threading.py", line 916, in _bootstrap_inner
    self.run()
  File "/usr/local/lib/python3.6/site-packages/anyio/_backends/_asyncio.py", line 867, in run
    result = context.run(func, *args)
  File "/usr/local/lib/python3.6/site-packages/contextvars/__init__.py", line 38, in run
    return callable(*args, **kwargs)
  File "./main.py", line 616, in convert_markdown
    initialize_model()
  File "./main.py", line 78, in initialize_model
    style_embedding_size=256
  File "./train.py", line 56, in __init__
    super(StyleSynthesisModel, self).__init__(reader=None, **kwargs)
  File "./tf_base_model.py", line 110, in __init__
    self.graph = self.build_graph()
  File "./tf_base_model.py", line 401, in build_graph
    self.loss = self.calculate_loss()
  File "./train.py", line 81, in calculate_loss
    style_emb = style_encoder(self.ref_x)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/engine/topology.py", line 258, in __call__
    output = super(Layer, self).__call__(inputs, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/layers/base.py", line 696, in __call__
    outputs = self.call(inputs, *args, **kwargs)
  File "./train.py", line 26, in call
    x = self.bi_rnn(stroke_sequence)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/engine/topology.py", line 258, in __call__
    output = super(Layer, self).__call__(inputs, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/layers/base.py", line 696, in __call__
    outputs = self.call(inputs, *args, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/layers/wrappers.py", line 347, in call
    y_rev = self.backward_layer.call(inputs, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/layers/recurrent.py", line 2056, in call
    inputs, mask=mask, training=training, initial_state=initial_state)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/layers/recurrent.py", line 613, in call
    input_length=timesteps)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/backend.py", line 2963, in rnn
    outputs = output_ta.stack()
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/ops/tensor_array_ops.py", line 893, in stack
    return self._implementation.stack(name=name)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/ops/tensor_array_ops.py", line 291, in stack
    return self.gather(math_ops.range(0, self.size()), name=name)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/ops/tensor_array_ops.py", line 305, in gather
    element_shape=element_shape)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/ops/gen_data_flow_ops.py", line 4186, in _tensor_array_gather_v3
    flow_in=flow_in, dtype=dtype, element_shape=element_shape, name=name)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/framework/op_def_library.py", line 787, in _apply_op_helper
    op_def=op_def)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/framework/ops.py", line 3271, in create_op
    op_def=op_def)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/framework/ops.py", line 1650, in __init__
    self._traceback = self._graph._extract_stack()  # pylint: disable=protected-access

UnimplementedError (see above for traceback): TensorArray has size zero, but element shape [?,128] is not fully defined. Currently only static shapes are supported when packing zero-size TensorArrays.
	 [[Node: style_encoder/bidirectional/TensorArrayStack_1/TensorArrayGatherV3 = TensorArrayGatherV3[_class=["loc:@style_encoder/bidirectional/TensorArray_2"], dtype=DT_FLOAT, element_shape=[?,128], _device="/job:localhost/replica:0/task:0/device:CPU:0"](style_encoder/bidirectional/TensorArray_2, style_encoder/bidirectional/TensorArrayStack_1/range, style_encoder/bidirectional/while_1/Exit_1)]]

[[05/11/2025 09:13:41 AM]] Processing failed for item 7: TensorArray has size zero, but element shape [?,128] is not fully defined. Currently only static shapes are supported when packing zero-size TensorArrays.
	 [[Node: style_encoder/bidirectional/TensorArrayStack_1/TensorArrayGatherV3 = TensorArrayGatherV3[_class=["loc:@style_encoder/bidirectional/TensorArray_2"], dtype=DT_FLOAT, element_shape=[?,128], _device="/job:localhost/replica:0/task:0/device:CPU:0"](style_encoder/bidirectional/TensorArray_2, style_encoder/bidirectional/TensorArrayStack_1/range, style_encoder/bidirectional/while_1/Exit_1)]]

Caused by op 'style_encoder/bidirectional/TensorArrayStack_1/TensorArrayGatherV3', defined at:
  File "/usr/local/lib/python3.6/threading.py", line 884, in _bootstrap
    self._bootstrap_inner()
  File "/usr/local/lib/python3.6/threading.py", line 916, in _bootstrap_inner
    self.run()
  File "/usr/local/lib/python3.6/site-packages/anyio/_backends/_asyncio.py", line 867, in run
    result = context.run(func, *args)
  File "/usr/local/lib/python3.6/site-packages/contextvars/__init__.py", line 38, in run
    return callable(*args, **kwargs)
  File "./main.py", line 616, in convert_markdown
    initialize_model()
  File "./main.py", line 78, in initialize_model
    style_embedding_size=256
  File "./train.py", line 56, in __init__
    super(StyleSynthesisModel, self).__init__(reader=None, **kwargs)
  File "./tf_base_model.py", line 110, in __init__
    self.graph = self.build_graph()
  File "./tf_base_model.py", line 401, in build_graph
    self.loss = self.calculate_loss()
  File "./train.py", line 81, in calculate_loss
    style_emb = style_encoder(self.ref_x)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/engine/topology.py", line 258, in __call__
    output = super(Layer, self).__call__(inputs, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/layers/base.py", line 696, in __call__
    outputs = self.call(inputs, *args, **kwargs)
  File "./train.py", line 26, in call
    x = self.bi_rnn(stroke_sequence)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/engine/topology.py", line 258, in __call__
    output = super(Layer, self).__call__(inputs, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/layers/base.py", line 696, in __call__
    outputs = self.call(inputs, *args, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/layers/wrappers.py", line 347, in call
    y_rev = self.backward_layer.call(inputs, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/layers/recurrent.py", line 2056, in call
    inputs, mask=mask, training=training, initial_state=initial_state)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/layers/recurrent.py", line 613, in call
    input_length=timesteps)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/backend.py", line 2963, in rnn
    outputs = output_ta.stack()
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/ops/tensor_array_ops.py", line 893, in stack
    return self._implementation.stack(name=name)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/ops/tensor_array_ops.py", line 291, in stack
    return self.gather(math_ops.range(0, self.size()), name=name)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/ops/tensor_array_ops.py", line 305, in gather
    element_shape=element_shape)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/ops/gen_data_flow_ops.py", line 4186, in _tensor_array_gather_v3
    flow_in=flow_in, dtype=dtype, element_shape=element_shape, name=name)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/framework/op_def_library.py", line 787, in _apply_op_helper
    op_def=op_def)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/framework/ops.py", line 3271, in create_op
    op_def=op_def)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/framework/ops.py", line 1650, in __init__
    self._traceback = self._graph._extract_stack()  # pylint: disable=protected-access

UnimplementedError (see above for traceback): TensorArray has size zero, but element shape [?,128] is not fully defined. Currently only static shapes are supported when packing zero-size TensorArrays.
	 [[Node: style_encoder/bidirectional/TensorArrayStack_1/TensorArrayGatherV3 = TensorArrayGatherV3[_class=["loc:@style_encoder/bidirectional/TensorArray_2"], dtype=DT_FLOAT, element_shape=[?,128], _device="/job:localhost/replica:0/task:0/device:CPU:0"](style_encoder/bidirectional/TensorArray_2, style_encoder/bidirectional/TensorArrayStack_1/range, style_encoder/bidirectional/while_1/Exit_1)]]

[[05/11/2025 09:13:41 AM]] Error during model inference: TensorArray has size zero, but element shape [?,128] is not fully defined. Currently only static shapes are supported when packing zero-size TensorArrays.
	 [[Node: style_encoder/bidirectional/TensorArrayStack/TensorArrayGatherV3 = TensorArrayGatherV3[_class=["loc:@style_encoder/bidirectional/TensorArray"], dtype=DT_FLOAT, element_shape=[?,128], _device="/job:localhost/replica:0/task:0/device:CPU:0"](style_encoder/bidirectional/TensorArray, style_encoder/bidirectional/TensorArrayStack/range, style_encoder/bidirectional/while/Exit_1)]]

Caused by op 'style_encoder/bidirectional/TensorArrayStack/TensorArrayGatherV3', defined at:
  File "/usr/local/lib/python3.6/threading.py", line 884, in _bootstrap
    self._bootstrap_inner()
  File "/usr/local/lib/python3.6/threading.py", line 916, in _bootstrap_inner
    self.run()
  File "/usr/local/lib/python3.6/site-packages/anyio/_backends/_asyncio.py", line 867, in run
    result = context.run(func, *args)
  File "/usr/local/lib/python3.6/site-packages/contextvars/__init__.py", line 38, in run
    return callable(*args, **kwargs)
  File "./main.py", line 616, in convert_markdown
    initialize_model()
  File "./main.py", line 78, in initialize_model
    style_embedding_size=256
  File "./train.py", line 56, in __init__
    super(StyleSynthesisModel, self).__init__(reader=None, **kwargs)
  File "./tf_base_model.py", line 110, in __init__
    self.graph = self.build_graph()
  File "./tf_base_model.py", line 401, in build_graph
    self.loss = self.calculate_loss()
  File "./train.py", line 81, in calculate_loss
    style_emb = style_encoder(self.ref_x)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/engine/topology.py", line 258, in __call__
    output = super(Layer, self).__call__(inputs, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/layers/base.py", line 696, in __call__
    outputs = self.call(inputs, *args, **kwargs)
  File "./train.py", line 26, in call
    x = self.bi_rnn(stroke_sequence)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/engine/topology.py", line 258, in __call__
    output = super(Layer, self).__call__(inputs, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/layers/base.py", line 696, in __call__
    outputs = self.call(inputs, *args, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/layers/wrappers.py", line 346, in call
    y = self.forward_layer.call(inputs, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/layers/recurrent.py", line 2056, in call
    inputs, mask=mask, training=training, initial_state=initial_state)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/layers/recurrent.py", line 613, in call
    input_length=timesteps)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/backend.py", line 2963, in rnn
    outputs = output_ta.stack()
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/ops/tensor_array_ops.py", line 893, in stack
    return self._implementation.stack(name=name)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/ops/tensor_array_ops.py", line 291, in stack
    return self.gather(math_ops.range(0, self.size()), name=name)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/ops/tensor_array_ops.py", line 305, in gather
    element_shape=element_shape)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/ops/gen_data_flow_ops.py", line 4186, in _tensor_array_gather_v3
    flow_in=flow_in, dtype=dtype, element_shape=element_shape, name=name)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/framework/op_def_library.py", line 787, in _apply_op_helper
    op_def=op_def)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/framework/ops.py", line 3271, in create_op
    op_def=op_def)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/framework/ops.py", line 1650, in __init__
    self._traceback = self._graph._extract_stack()  # pylint: disable=protected-access

UnimplementedError (see above for traceback): TensorArray has size zero, but element shape [?,128] is not fully defined. Currently only static shapes are supported when packing zero-size TensorArrays.
	 [[Node: style_encoder/bidirectional/TensorArrayStack/TensorArrayGatherV3 = TensorArrayGatherV3[_class=["loc:@style_encoder/bidirectional/TensorArray"], dtype=DT_FLOAT, element_shape=[?,128], _device="/job:localhost/replica:0/task:0/device:CPU:0"](style_encoder/bidirectional/TensorArray, style_encoder/bidirectional/TensorArrayStack/range, style_encoder/bidirectional/while/Exit_1)]]

[[05/11/2025 09:13:41 AM]] Processing failed for item 8: TensorArray has size zero, but element shape [?,128] is not fully defined. Currently only static shapes are supported when packing zero-size TensorArrays.
	 [[Node: style_encoder/bidirectional/TensorArrayStack/TensorArrayGatherV3 = TensorArrayGatherV3[_class=["loc:@style_encoder/bidirectional/TensorArray"], dtype=DT_FLOAT, element_shape=[?,128], _device="/job:localhost/replica:0/task:0/device:CPU:0"](style_encoder/bidirectional/TensorArray, style_encoder/bidirectional/TensorArrayStack/range, style_encoder/bidirectional/while/Exit_1)]]

Caused by op 'style_encoder/bidirectional/TensorArrayStack/TensorArrayGatherV3', defined at:
  File "/usr/local/lib/python3.6/threading.py", line 884, in _bootstrap
    self._bootstrap_inner()
  File "/usr/local/lib/python3.6/threading.py", line 916, in _bootstrap_inner
    self.run()
  File "/usr/local/lib/python3.6/site-packages/anyio/_backends/_asyncio.py", line 867, in run
    result = context.run(func, *args)
  File "/usr/local/lib/python3.6/site-packages/contextvars/__init__.py", line 38, in run
    return callable(*args, **kwargs)
  File "./main.py", line 616, in convert_markdown
    initialize_model()
  File "./main.py", line 78, in initialize_model
    style_embedding_size=256
  File "./train.py", line 56, in __init__
    super(StyleSynthesisModel, self).__init__(reader=None, **kwargs)
  File "./tf_base_model.py", line 110, in __init__
    self.graph = self.build_graph()
  File "./tf_base_model.py", line 401, in build_graph
    self.loss = self.calculate_loss()
  File "./train.py", line 81, in calculate_loss
    style_emb = style_encoder(self.ref_x)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/engine/topology.py", line 258, in __call__
    output = super(Layer, self).__call__(inputs, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/layers/base.py", line 696, in __call__
    outputs = self.call(inputs, *args, **kwargs)
  File "./train.py", line 26, in call
    x = self.bi_rnn(stroke_sequence)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/engine/topology.py", line 258, in __call__
    output = super(Layer, self).__call__(inputs, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/layers/base.py", line 696, in __call__
    outputs = self.call(inputs, *args, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/layers/wrappers.py", line 346, in call
    y = self.forward_layer.call(inputs, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/layers/recurrent.py", line 2056, in call
    inputs, mask=mask, training=training, initial_state=initial_state)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/layers/recurrent.py", line 613, in call
    input_length=timesteps)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/backend.py", line 2963, in rnn
    outputs = output_ta.stack()
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/ops/tensor_array_ops.py", line 893, in stack
    return self._implementation.stack(name=name)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/ops/tensor_array_ops.py", line 291, in stack
    return self.gather(math_ops.range(0, self.size()), name=name)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/ops/tensor_array_ops.py", line 305, in gather
    element_shape=element_shape)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/ops/gen_data_flow_ops.py", line 4186, in _tensor_array_gather_v3
    flow_in=flow_in, dtype=dtype, element_shape=element_shape, name=name)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/framework/op_def_library.py", line 787, in _apply_op_helper
    op_def=op_def)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/framework/ops.py", line 3271, in create_op
    op_def=op_def)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/framework/ops.py", line 1650, in __init__
    self._traceback = self._graph._extract_stack()  # pylint: disable=protected-access

UnimplementedError (see above for traceback): TensorArray has size zero, but element shape [?,128] is not fully defined. Currently only static shapes are supported when packing zero-size TensorArrays.
	 [[Node: style_encoder/bidirectional/TensorArrayStack/TensorArrayGatherV3 = TensorArrayGatherV3[_class=["loc:@style_encoder/bidirectional/TensorArray"], dtype=DT_FLOAT, element_shape=[?,128], _device="/job:localhost/replica:0/task:0/device:CPU:0"](style_encoder/bidirectional/TensorArray, style_encoder/bidirectional/TensorArrayStack/range, style_encoder/bidirectional/while/Exit_1)]]

[[05/11/2025 09:13:41 AM]] Error during model inference: TensorArray has size zero, but element shape [?,128] is not fully defined. Currently only static shapes are supported when packing zero-size TensorArrays.
	 [[Node: style_encoder/bidirectional/TensorArrayStack_1/TensorArrayGatherV3 = TensorArrayGatherV3[_class=["loc:@style_encoder/bidirectional/TensorArray_2"], dtype=DT_FLOAT, element_shape=[?,128], _device="/job:localhost/replica:0/task:0/device:CPU:0"](style_encoder/bidirectional/TensorArray_2, style_encoder/bidirectional/TensorArrayStack_1/range, style_encoder/bidirectional/while_1/Exit_1)]]

Caused by op 'style_encoder/bidirectional/TensorArrayStack_1/TensorArrayGatherV3', defined at:
  File "/usr/local/lib/python3.6/threading.py", line 884, in _bootstrap
    self._bootstrap_inner()
  File "/usr/local/lib/python3.6/threading.py", line 916, in _bootstrap_inner
    self.run()
  File "/usr/local/lib/python3.6/site-packages/anyio/_backends/_asyncio.py", line 867, in run
    result = context.run(func, *args)
  File "/usr/local/lib/python3.6/site-packages/contextvars/__init__.py", line 38, in run
    return callable(*args, **kwargs)
  File "./main.py", line 616, in convert_markdown
    initialize_model()
  File "./main.py", line 78, in initialize_model
    style_embedding_size=256
  File "./train.py", line 56, in __init__
    super(StyleSynthesisModel, self).__init__(reader=None, **kwargs)
  File "./tf_base_model.py", line 110, in __init__
    self.graph = self.build_graph()
  File "./tf_base_model.py", line 401, in build_graph
    self.loss = self.calculate_loss()
  File "./train.py", line 81, in calculate_loss
    style_emb = style_encoder(self.ref_x)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/engine/topology.py", line 258, in __call__
    output = super(Layer, self).__call__(inputs, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/layers/base.py", line 696, in __call__
    outputs = self.call(inputs, *args, **kwargs)
  File "./train.py", line 26, in call
    x = self.bi_rnn(stroke_sequence)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/engine/topology.py", line 258, in __call__
    output = super(Layer, self).__call__(inputs, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/layers/base.py", line 696, in __call__
    outputs = self.call(inputs, *args, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/layers/wrappers.py", line 347, in call
    y_rev = self.backward_layer.call(inputs, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/layers/recurrent.py", line 2056, in call
    inputs, mask=mask, training=training, initial_state=initial_state)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/layers/recurrent.py", line 613, in call
    input_length=timesteps)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/backend.py", line 2963, in rnn
    outputs = output_ta.stack()
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/ops/tensor_array_ops.py", line 893, in stack
    return self._implementation.stack(name=name)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/ops/tensor_array_ops.py", line 291, in stack
    return self.gather(math_ops.range(0, self.size()), name=name)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/ops/tensor_array_ops.py", line 305, in gather
    element_shape=element_shape)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/ops/gen_data_flow_ops.py", line 4186, in _tensor_array_gather_v3
    flow_in=flow_in, dtype=dtype, element_shape=element_shape, name=name)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/framework/op_def_library.py", line 787, in _apply_op_helper
    op_def=op_def)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/framework/ops.py", line 3271, in create_op
    op_def=op_def)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/framework/ops.py", line 1650, in __init__
    self._traceback = self._graph._extract_stack()  # pylint: disable=protected-access

UnimplementedError (see above for traceback): TensorArray has size zero, but element shape [?,128] is not fully defined. Currently only static shapes are supported when packing zero-size TensorArrays.
	 [[Node: style_encoder/bidirectional/TensorArrayStack_1/TensorArrayGatherV3 = TensorArrayGatherV3[_class=["loc:@style_encoder/bidirectional/TensorArray_2"], dtype=DT_FLOAT, element_shape=[?,128], _device="/job:localhost/replica:0/task:0/device:CPU:0"](style_encoder/bidirectional/TensorArray_2, style_encoder/bidirectional/TensorArrayStack_1/range, style_encoder/bidirectional/while_1/Exit_1)]]

[[05/11/2025 09:13:41 AM]] Processing failed for item 9: TensorArray has size zero, but element shape [?,128] is not fully defined. Currently only static shapes are supported when packing zero-size TensorArrays.
	 [[Node: style_encoder/bidirectional/TensorArrayStack_1/TensorArrayGatherV3 = TensorArrayGatherV3[_class=["loc:@style_encoder/bidirectional/TensorArray_2"], dtype=DT_FLOAT, element_shape=[?,128], _device="/job:localhost/replica:0/task:0/device:CPU:0"](style_encoder/bidirectional/TensorArray_2, style_encoder/bidirectional/TensorArrayStack_1/range, style_encoder/bidirectional/while_1/Exit_1)]]

Caused by op 'style_encoder/bidirectional/TensorArrayStack_1/TensorArrayGatherV3', defined at:
  File "/usr/local/lib/python3.6/threading.py", line 884, in _bootstrap
    self._bootstrap_inner()
  File "/usr/local/lib/python3.6/threading.py", line 916, in _bootstrap_inner
    self.run()
  File "/usr/local/lib/python3.6/site-packages/anyio/_backends/_asyncio.py", line 867, in run
    result = context.run(func, *args)
  File "/usr/local/lib/python3.6/site-packages/contextvars/__init__.py", line 38, in run
    return callable(*args, **kwargs)
  File "./main.py", line 616, in convert_markdown
    initialize_model()
  File "./main.py", line 78, in initialize_model
    style_embedding_size=256
  File "./train.py", line 56, in __init__
    super(StyleSynthesisModel, self).__init__(reader=None, **kwargs)
  File "./tf_base_model.py", line 110, in __init__
    self.graph = self.build_graph()
  File "./tf_base_model.py", line 401, in build_graph
    self.loss = self.calculate_loss()
  File "./train.py", line 81, in calculate_loss
    style_emb = style_encoder(self.ref_x)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/engine/topology.py", line 258, in __call__
    output = super(Layer, self).__call__(inputs, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/layers/base.py", line 696, in __call__
    outputs = self.call(inputs, *args, **kwargs)
  File "./train.py", line 26, in call
    x = self.bi_rnn(stroke_sequence)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/engine/topology.py", line 258, in __call__
    output = super(Layer, self).__call__(inputs, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/layers/base.py", line 696, in __call__
    outputs = self.call(inputs, *args, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/layers/wrappers.py", line 347, in call
    y_rev = self.backward_layer.call(inputs, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/layers/recurrent.py", line 2056, in call
    inputs, mask=mask, training=training, initial_state=initial_state)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/layers/recurrent.py", line 613, in call
    input_length=timesteps)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/backend.py", line 2963, in rnn
    outputs = output_ta.stack()
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/ops/tensor_array_ops.py", line 893, in stack
    return self._implementation.stack(name=name)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/ops/tensor_array_ops.py", line 291, in stack
    return self.gather(math_ops.range(0, self.size()), name=name)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/ops/tensor_array_ops.py", line 305, in gather
    element_shape=element_shape)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/ops/gen_data_flow_ops.py", line 4186, in _tensor_array_gather_v3
    flow_in=flow_in, dtype=dtype, element_shape=element_shape, name=name)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/framework/op_def_library.py", line 787, in _apply_op_helper
    op_def=op_def)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/framework/ops.py", line 3271, in create_op
    op_def=op_def)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/framework/ops.py", line 1650, in __init__
    self._traceback = self._graph._extract_stack()  # pylint: disable=protected-access

UnimplementedError (see above for traceback): TensorArray has size zero, but element shape [?,128] is not fully defined. Currently only static shapes are supported when packing zero-size TensorArrays.
	 [[Node: style_encoder/bidirectional/TensorArrayStack_1/TensorArrayGatherV3 = TensorArrayGatherV3[_class=["loc:@style_encoder/bidirectional/TensorArray_2"], dtype=DT_FLOAT, element_shape=[?,128], _device="/job:localhost/replica:0/task:0/device:CPU:0"](style_encoder/bidirectional/TensorArray_2, style_encoder/bidirectional/TensorArrayStack_1/range, style_encoder/bidirectional/while_1/Exit_1)]]

[[05/11/2025 09:13:41 AM]] Error during model inference: TensorArray has size zero, but element shape [?,128] is not fully defined. Currently only static shapes are supported when packing zero-size TensorArrays.
	 [[Node: style_encoder/bidirectional/TensorArrayStack/TensorArrayGatherV3 = TensorArrayGatherV3[_class=["loc:@style_encoder/bidirectional/TensorArray"], dtype=DT_FLOAT, element_shape=[?,128], _device="/job:localhost/replica:0/task:0/device:CPU:0"](style_encoder/bidirectional/TensorArray, style_encoder/bidirectional/TensorArrayStack/range, style_encoder/bidirectional/while/Exit_1)]]

Caused by op 'style_encoder/bidirectional/TensorArrayStack/TensorArrayGatherV3', defined at:
  File "/usr/local/lib/python3.6/threading.py", line 884, in _bootstrap
    self._bootstrap_inner()
  File "/usr/local/lib/python3.6/threading.py", line 916, in _bootstrap_inner
    self.run()
  File "/usr/local/lib/python3.6/site-packages/anyio/_backends/_asyncio.py", line 867, in run
    result = context.run(func, *args)
  File "/usr/local/lib/python3.6/site-packages/contextvars/__init__.py", line 38, in run
    return callable(*args, **kwargs)
  File "./main.py", line 616, in convert_markdown
    initialize_model()
  File "./main.py", line 78, in initialize_model
    style_embedding_size=256
  File "./train.py", line 56, in __init__
    super(StyleSynthesisModel, self).__init__(reader=None, **kwargs)
  File "./tf_base_model.py", line 110, in __init__
    self.graph = self.build_graph()
  File "./tf_base_model.py", line 401, in build_graph
    self.loss = self.calculate_loss()
  File "./train.py", line 81, in calculate_loss
    style_emb = style_encoder(self.ref_x)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/engine/topology.py", line 258, in __call__
    output = super(Layer, self).__call__(inputs, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/layers/base.py", line 696, in __call__
    outputs = self.call(inputs, *args, **kwargs)
  File "./train.py", line 26, in call
    x = self.bi_rnn(stroke_sequence)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/engine/topology.py", line 258, in __call__
    output = super(Layer, self).__call__(inputs, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/layers/base.py", line 696, in __call__
    outputs = self.call(inputs, *args, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/layers/wrappers.py", line 346, in call
    y = self.forward_layer.call(inputs, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/layers/recurrent.py", line 2056, in call
    inputs, mask=mask, training=training, initial_state=initial_state)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/layers/recurrent.py", line 613, in call
    input_length=timesteps)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/backend.py", line 2963, in rnn
    outputs = output_ta.stack()
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/ops/tensor_array_ops.py", line 893, in stack
    return self._implementation.stack(name=name)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/ops/tensor_array_ops.py", line 291, in stack
    return self.gather(math_ops.range(0, self.size()), name=name)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/ops/tensor_array_ops.py", line 305, in gather
    element_shape=element_shape)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/ops/gen_data_flow_ops.py", line 4186, in _tensor_array_gather_v3
    flow_in=flow_in, dtype=dtype, element_shape=element_shape, name=name)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/framework/op_def_library.py", line 787, in _apply_op_helper
    op_def=op_def)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/framework/ops.py", line 3271, in create_op
    op_def=op_def)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/framework/ops.py", line 1650, in __init__
    self._traceback = self._graph._extract_stack()  # pylint: disable=protected-access

UnimplementedError (see above for traceback): TensorArray has size zero, but element shape [?,128] is not fully defined. Currently only static shapes are supported when packing zero-size TensorArrays.
	 [[Node: style_encoder/bidirectional/TensorArrayStack/TensorArrayGatherV3 = TensorArrayGatherV3[_class=["loc:@style_encoder/bidirectional/TensorArray"], dtype=DT_FLOAT, element_shape=[?,128], _device="/job:localhost/replica:0/task:0/device:CPU:0"](style_encoder/bidirectional/TensorArray, style_encoder/bidirectional/TensorArrayStack/range, style_encoder/bidirectional/while/Exit_1)]]

[[05/11/2025 09:13:41 AM]] Processing failed for item 10: TensorArray has size zero, but element shape [?,128] is not fully defined. Currently only static shapes are supported when packing zero-size TensorArrays.
	 [[Node: style_encoder/bidirectional/TensorArrayStack/TensorArrayGatherV3 = TensorArrayGatherV3[_class=["loc:@style_encoder/bidirectional/TensorArray"], dtype=DT_FLOAT, element_shape=[?,128], _device="/job:localhost/replica:0/task:0/device:CPU:0"](style_encoder/bidirectional/TensorArray, style_encoder/bidirectional/TensorArrayStack/range, style_encoder/bidirectional/while/Exit_1)]]

Caused by op 'style_encoder/bidirectional/TensorArrayStack/TensorArrayGatherV3', defined at:
  File "/usr/local/lib/python3.6/threading.py", line 884, in _bootstrap
    self._bootstrap_inner()
  File "/usr/local/lib/python3.6/threading.py", line 916, in _bootstrap_inner
    self.run()
  File "/usr/local/lib/python3.6/site-packages/anyio/_backends/_asyncio.py", line 867, in run
    result = context.run(func, *args)
  File "/usr/local/lib/python3.6/site-packages/contextvars/__init__.py", line 38, in run
    return callable(*args, **kwargs)
  File "./main.py", line 616, in convert_markdown
    initialize_model()
  File "./main.py", line 78, in initialize_model
    style_embedding_size=256
  File "./train.py", line 56, in __init__
    super(StyleSynthesisModel, self).__init__(reader=None, **kwargs)
  File "./tf_base_model.py", line 110, in __init__
    self.graph = self.build_graph()
  File "./tf_base_model.py", line 401, in build_graph
    self.loss = self.calculate_loss()
  File "./train.py", line 81, in calculate_loss
    style_emb = style_encoder(self.ref_x)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/engine/topology.py", line 258, in __call__
    output = super(Layer, self).__call__(inputs, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/layers/base.py", line 696, in __call__
    outputs = self.call(inputs, *args, **kwargs)
  File "./train.py", line 26, in call
    x = self.bi_rnn(stroke_sequence)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/engine/topology.py", line 258, in __call__
    output = super(Layer, self).__call__(inputs, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/layers/base.py", line 696, in __call__
    outputs = self.call(inputs, *args, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/layers/wrappers.py", line 346, in call
    y = self.forward_layer.call(inputs, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/layers/recurrent.py", line 2056, in call
    inputs, mask=mask, training=training, initial_state=initial_state)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/layers/recurrent.py", line 613, in call
    input_length=timesteps)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/backend.py", line 2963, in rnn
    outputs = output_ta.stack()
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/ops/tensor_array_ops.py", line 893, in stack
    return self._implementation.stack(name=name)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/ops/tensor_array_ops.py", line 291, in stack
    return self.gather(math_ops.range(0, self.size()), name=name)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/ops/tensor_array_ops.py", line 305, in gather
    element_shape=element_shape)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/ops/gen_data_flow_ops.py", line 4186, in _tensor_array_gather_v3
    flow_in=flow_in, dtype=dtype, element_shape=element_shape, name=name)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/framework/op_def_library.py", line 787, in _apply_op_helper
    op_def=op_def)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/framework/ops.py", line 3271, in create_op
    op_def=op_def)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/framework/ops.py", line 1650, in __init__
    self._traceback = self._graph._extract_stack()  # pylint: disable=protected-access

UnimplementedError (see above for traceback): TensorArray has size zero, but element shape [?,128] is not fully defined. Currently only static shapes are supported when packing zero-size TensorArrays.
	 [[Node: style_encoder/bidirectional/TensorArrayStack/TensorArrayGatherV3 = TensorArrayGatherV3[_class=["loc:@style_encoder/bidirectional/TensorArray"], dtype=DT_FLOAT, element_shape=[?,128], _device="/job:localhost/replica:0/task:0/device:CPU:0"](style_encoder/bidirectional/TensorArray, style_encoder/bidirectional/TensorArrayStack/range, style_encoder/bidirectional/while/Exit_1)]]

[[05/11/2025 09:13:41 AM]] Error during model inference: TensorArray has size zero, but element shape [?,128] is not fully defined. Currently only static shapes are supported when packing zero-size TensorArrays.
	 [[Node: style_encoder/bidirectional/TensorArrayStack_1/TensorArrayGatherV3 = TensorArrayGatherV3[_class=["loc:@style_encoder/bidirectional/TensorArray_2"], dtype=DT_FLOAT, element_shape=[?,128], _device="/job:localhost/replica:0/task:0/device:CPU:0"](style_encoder/bidirectional/TensorArray_2, style_encoder/bidirectional/TensorArrayStack_1/range, style_encoder/bidirectional/while_1/Exit_1)]]

Caused by op 'style_encoder/bidirectional/TensorArrayStack_1/TensorArrayGatherV3', defined at:
  File "/usr/local/lib/python3.6/threading.py", line 884, in _bootstrap
    self._bootstrap_inner()
  File "/usr/local/lib/python3.6/threading.py", line 916, in _bootstrap_inner
    self.run()
  File "/usr/local/lib/python3.6/site-packages/anyio/_backends/_asyncio.py", line 867, in run
    result = context.run(func, *args)
  File "/usr/local/lib/python3.6/site-packages/contextvars/__init__.py", line 38, in run
    return callable(*args, **kwargs)
  File "./main.py", line 616, in convert_markdown
    initialize_model()
  File "./main.py", line 78, in initialize_model
    style_embedding_size=256
  File "./train.py", line 56, in __init__
    super(StyleSynthesisModel, self).__init__(reader=None, **kwargs)
  File "./tf_base_model.py", line 110, in __init__
    self.graph = self.build_graph()
  File "./tf_base_model.py", line 401, in build_graph
    self.loss = self.calculate_loss()
  File "./train.py", line 81, in calculate_loss
    style_emb = style_encoder(self.ref_x)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/engine/topology.py", line 258, in __call__
    output = super(Layer, self).__call__(inputs, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/layers/base.py", line 696, in __call__
    outputs = self.call(inputs, *args, **kwargs)
  File "./train.py", line 26, in call
    x = self.bi_rnn(stroke_sequence)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/engine/topology.py", line 258, in __call__
    output = super(Layer, self).__call__(inputs, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/layers/base.py", line 696, in __call__
    outputs = self.call(inputs, *args, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/layers/wrappers.py", line 347, in call
    y_rev = self.backward_layer.call(inputs, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/layers/recurrent.py", line 2056, in call
    inputs, mask=mask, training=training, initial_state=initial_state)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/layers/recurrent.py", line 613, in call
    input_length=timesteps)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/backend.py", line 2963, in rnn
    outputs = output_ta.stack()
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/ops/tensor_array_ops.py", line 893, in stack
    return self._implementation.stack(name=name)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/ops/tensor_array_ops.py", line 291, in stack
    return self.gather(math_ops.range(0, self.size()), name=name)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/ops/tensor_array_ops.py", line 305, in gather
    element_shape=element_shape)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/ops/gen_data_flow_ops.py", line 4186, in _tensor_array_gather_v3
    flow_in=flow_in, dtype=dtype, element_shape=element_shape, name=name)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/framework/op_def_library.py", line 787, in _apply_op_helper
    op_def=op_def)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/framework/ops.py", line 3271, in create_op
    op_def=op_def)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/framework/ops.py", line 1650, in __init__
    self._traceback = self._graph._extract_stack()  # pylint: disable=protected-access

UnimplementedError (see above for traceback): TensorArray has size zero, but element shape [?,128] is not fully defined. Currently only static shapes are supported when packing zero-size TensorArrays.
	 [[Node: style_encoder/bidirectional/TensorArrayStack_1/TensorArrayGatherV3 = TensorArrayGatherV3[_class=["loc:@style_encoder/bidirectional/TensorArray_2"], dtype=DT_FLOAT, element_shape=[?,128], _device="/job:localhost/replica:0/task:0/device:CPU:0"](style_encoder/bidirectional/TensorArray_2, style_encoder/bidirectional/TensorArrayStack_1/range, style_encoder/bidirectional/while_1/Exit_1)]]

[[05/11/2025 09:13:41 AM]] Processing failed for item 12: TensorArray has size zero, but element shape [?,128] is not fully defined. Currently only static shapes are supported when packing zero-size TensorArrays.
	 [[Node: style_encoder/bidirectional/TensorArrayStack_1/TensorArrayGatherV3 = TensorArrayGatherV3[_class=["loc:@style_encoder/bidirectional/TensorArray_2"], dtype=DT_FLOAT, element_shape=[?,128], _device="/job:localhost/replica:0/task:0/device:CPU:0"](style_encoder/bidirectional/TensorArray_2, style_encoder/bidirectional/TensorArrayStack_1/range, style_encoder/bidirectional/while_1/Exit_1)]]

Caused by op 'style_encoder/bidirectional/TensorArrayStack_1/TensorArrayGatherV3', defined at:
  File "/usr/local/lib/python3.6/threading.py", line 884, in _bootstrap
    self._bootstrap_inner()
  File "/usr/local/lib/python3.6/threading.py", line 916, in _bootstrap_inner
    self.run()
  File "/usr/local/lib/python3.6/site-packages/anyio/_backends/_asyncio.py", line 867, in run
    result = context.run(func, *args)
  File "/usr/local/lib/python3.6/site-packages/contextvars/__init__.py", line 38, in run
    return callable(*args, **kwargs)
  File "./main.py", line 616, in convert_markdown
    initialize_model()
  File "./main.py", line 78, in initialize_model
    style_embedding_size=256
  File "./train.py", line 56, in __init__
    super(StyleSynthesisModel, self).__init__(reader=None, **kwargs)
  File "./tf_base_model.py", line 110, in __init__
    self.graph = self.build_graph()
  File "./tf_base_model.py", line 401, in build_graph
    self.loss = self.calculate_loss()
  File "./train.py", line 81, in calculate_loss
    style_emb = style_encoder(self.ref_x)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/engine/topology.py", line 258, in __call__
    output = super(Layer, self).__call__(inputs, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/layers/base.py", line 696, in __call__
    outputs = self.call(inputs, *args, **kwargs)
  File "./train.py", line 26, in call
    x = self.bi_rnn(stroke_sequence)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/engine/topology.py", line 258, in __call__
    output = super(Layer, self).__call__(inputs, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/layers/base.py", line 696, in __call__
    outputs = self.call(inputs, *args, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/layers/wrappers.py", line 347, in call
    y_rev = self.backward_layer.call(inputs, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/layers/recurrent.py", line 2056, in call
    inputs, mask=mask, training=training, initial_state=initial_state)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/layers/recurrent.py", line 613, in call
    input_length=timesteps)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/backend.py", line 2963, in rnn
    outputs = output_ta.stack()
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/ops/tensor_array_ops.py", line 893, in stack
    return self._implementation.stack(name=name)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/ops/tensor_array_ops.py", line 291, in stack
    return self.gather(math_ops.range(0, self.size()), name=name)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/ops/tensor_array_ops.py", line 305, in gather
    element_shape=element_shape)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/ops/gen_data_flow_ops.py", line 4186, in _tensor_array_gather_v3
    flow_in=flow_in, dtype=dtype, element_shape=element_shape, name=name)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/framework/op_def_library.py", line 787, in _apply_op_helper
    op_def=op_def)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/framework/ops.py", line 3271, in create_op
    op_def=op_def)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/framework/ops.py", line 1650, in __init__
    self._traceback = self._graph._extract_stack()  # pylint: disable=protected-access

UnimplementedError (see above for traceback): TensorArray has size zero, but element shape [?,128] is not fully defined. Currently only static shapes are supported when packing zero-size TensorArrays.
	 [[Node: style_encoder/bidirectional/TensorArrayStack_1/TensorArrayGatherV3 = TensorArrayGatherV3[_class=["loc:@style_encoder/bidirectional/TensorArray_2"], dtype=DT_FLOAT, element_shape=[?,128], _device="/job:localhost/replica:0/task:0/device:CPU:0"](style_encoder/bidirectional/TensorArray_2, style_encoder/bidirectional/TensorArrayStack_1/range, style_encoder/bidirectional/while_1/Exit_1)]]

[[05/11/2025 09:13:41 AM]] Error during model inference: TensorArray has size zero, but element shape [?,128] is not fully defined. Currently only static shapes are supported when packing zero-size TensorArrays.
	 [[Node: style_encoder/bidirectional/TensorArrayStack_1/TensorArrayGatherV3 = TensorArrayGatherV3[_class=["loc:@style_encoder/bidirectional/TensorArray_2"], dtype=DT_FLOAT, element_shape=[?,128], _device="/job:localhost/replica:0/task:0/device:CPU:0"](style_encoder/bidirectional/TensorArray_2, style_encoder/bidirectional/TensorArrayStack_1/range, style_encoder/bidirectional/while_1/Exit_1)]]

Caused by op 'style_encoder/bidirectional/TensorArrayStack_1/TensorArrayGatherV3', defined at:
  File "/usr/local/lib/python3.6/threading.py", line 884, in _bootstrap
    self._bootstrap_inner()
  File "/usr/local/lib/python3.6/threading.py", line 916, in _bootstrap_inner
    self.run()
  File "/usr/local/lib/python3.6/site-packages/anyio/_backends/_asyncio.py", line 867, in run
    result = context.run(func, *args)
  File "/usr/local/lib/python3.6/site-packages/contextvars/__init__.py", line 38, in run
    return callable(*args, **kwargs)
  File "./main.py", line 616, in convert_markdown
    initialize_model()
  File "./main.py", line 78, in initialize_model
    style_embedding_size=256
  File "./train.py", line 56, in __init__
    super(StyleSynthesisModel, self).__init__(reader=None, **kwargs)
  File "./tf_base_model.py", line 110, in __init__
    self.graph = self.build_graph()
  File "./tf_base_model.py", line 401, in build_graph
    self.loss = self.calculate_loss()
  File "./train.py", line 81, in calculate_loss
    style_emb = style_encoder(self.ref_x)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/engine/topology.py", line 258, in __call__
    output = super(Layer, self).__call__(inputs, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/layers/base.py", line 696, in __call__
    outputs = self.call(inputs, *args, **kwargs)
  File "./train.py", line 26, in call
    x = self.bi_rnn(stroke_sequence)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/engine/topology.py", line 258, in __call__
    output = super(Layer, self).__call__(inputs, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/layers/base.py", line 696, in __call__
    outputs = self.call(inputs, *args, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/layers/wrappers.py", line 347, in call
    y_rev = self.backward_layer.call(inputs, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/layers/recurrent.py", line 2056, in call
    inputs, mask=mask, training=training, initial_state=initial_state)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/layers/recurrent.py", line 613, in call
    input_length=timesteps)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/backend.py", line 2963, in rnn
    outputs = output_ta.stack()
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/ops/tensor_array_ops.py", line 893, in stack
    return self._implementation.stack(name=name)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/ops/tensor_array_ops.py", line 291, in stack
    return self.gather(math_ops.range(0, self.size()), name=name)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/ops/tensor_array_ops.py", line 305, in gather
    element_shape=element_shape)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/ops/gen_data_flow_ops.py", line 4186, in _tensor_array_gather_v3
    flow_in=flow_in, dtype=dtype, element_shape=element_shape, name=name)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/framework/op_def_library.py", line 787, in _apply_op_helper
    op_def=op_def)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/framework/ops.py", line 3271, in create_op
    op_def=op_def)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/framework/ops.py", line 1650, in __init__
    self._traceback = self._graph._extract_stack()  # pylint: disable=protected-access

UnimplementedError (see above for traceback): TensorArray has size zero, but element shape [?,128] is not fully defined. Currently only static shapes are supported when packing zero-size TensorArrays.
	 [[Node: style_encoder/bidirectional/TensorArrayStack_1/TensorArrayGatherV3 = TensorArrayGatherV3[_class=["loc:@style_encoder/bidirectional/TensorArray_2"], dtype=DT_FLOAT, element_shape=[?,128], _device="/job:localhost/replica:0/task:0/device:CPU:0"](style_encoder/bidirectional/TensorArray_2, style_encoder/bidirectional/TensorArrayStack_1/range, style_encoder/bidirectional/while_1/Exit_1)]]

[[05/11/2025 09:13:41 AM]] Processing failed for item 11: TensorArray has size zero, but element shape [?,128] is not fully defined. Currently only static shapes are supported when packing zero-size TensorArrays.
	 [[Node: style_encoder/bidirectional/TensorArrayStack_1/TensorArrayGatherV3 = TensorArrayGatherV3[_class=["loc:@style_encoder/bidirectional/TensorArray_2"], dtype=DT_FLOAT, element_shape=[?,128], _device="/job:localhost/replica:0/task:0/device:CPU:0"](style_encoder/bidirectional/TensorArray_2, style_encoder/bidirectional/TensorArrayStack_1/range, style_encoder/bidirectional/while_1/Exit_1)]]

Caused by op 'style_encoder/bidirectional/TensorArrayStack_1/TensorArrayGatherV3', defined at:
  File "/usr/local/lib/python3.6/threading.py", line 884, in _bootstrap
    self._bootstrap_inner()
  File "/usr/local/lib/python3.6/threading.py", line 916, in _bootstrap_inner
    self.run()
  File "/usr/local/lib/python3.6/site-packages/anyio/_backends/_asyncio.py", line 867, in run
    result = context.run(func, *args)
  File "/usr/local/lib/python3.6/site-packages/contextvars/__init__.py", line 38, in run
    return callable(*args, **kwargs)
  File "./main.py", line 616, in convert_markdown
    initialize_model()
  File "./main.py", line 78, in initialize_model
    style_embedding_size=256
  File "./train.py", line 56, in __init__
    super(StyleSynthesisModel, self).__init__(reader=None, **kwargs)
  File "./tf_base_model.py", line 110, in __init__
    self.graph = self.build_graph()
  File "./tf_base_model.py", line 401, in build_graph
    self.loss = self.calculate_loss()
  File "./train.py", line 81, in calculate_loss
    style_emb = style_encoder(self.ref_x)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/engine/topology.py", line 258, in __call__
    output = super(Layer, self).__call__(inputs, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/layers/base.py", line 696, in __call__
    outputs = self.call(inputs, *args, **kwargs)
  File "./train.py", line 26, in call
    x = self.bi_rnn(stroke_sequence)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/engine/topology.py", line 258, in __call__
    output = super(Layer, self).__call__(inputs, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/layers/base.py", line 696, in __call__
    outputs = self.call(inputs, *args, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/layers/wrappers.py", line 347, in call
    y_rev = self.backward_layer.call(inputs, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/layers/recurrent.py", line 2056, in call
    inputs, mask=mask, training=training, initial_state=initial_state)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/layers/recurrent.py", line 613, in call
    input_length=timesteps)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/backend.py", line 2963, in rnn
    outputs = output_ta.stack()
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/ops/tensor_array_ops.py", line 893, in stack
    return self._implementation.stack(name=name)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/ops/tensor_array_ops.py", line 291, in stack
    return self.gather(math_ops.range(0, self.size()), name=name)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/ops/tensor_array_ops.py", line 305, in gather
    element_shape=element_shape)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/ops/gen_data_flow_ops.py", line 4186, in _tensor_array_gather_v3
    flow_in=flow_in, dtype=dtype, element_shape=element_shape, name=name)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/framework/op_def_library.py", line 787, in _apply_op_helper
    op_def=op_def)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/framework/ops.py", line 3271, in create_op
    op_def=op_def)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/framework/ops.py", line 1650, in __init__
    self._traceback = self._graph._extract_stack()  # pylint: disable=protected-access

UnimplementedError (see above for traceback): TensorArray has size zero, but element shape [?,128] is not fully defined. Currently only static shapes are supported when packing zero-size TensorArrays.
	 [[Node: style_encoder/bidirectional/TensorArrayStack_1/TensorArrayGatherV3 = TensorArrayGatherV3[_class=["loc:@style_encoder/bidirectional/TensorArray_2"], dtype=DT_FLOAT, element_shape=[?,128], _device="/job:localhost/replica:0/task:0/device:CPU:0"](style_encoder/bidirectional/TensorArray_2, style_encoder/bidirectional/TensorArrayStack_1/range, style_encoder/bidirectional/while_1/Exit_1)]]

[[05/11/2025 09:13:41 AM]] Error during model inference: TensorArray has size zero, but element shape [?,128] is not fully defined. Currently only static shapes are supported when packing zero-size TensorArrays.
	 [[Node: style_encoder/bidirectional/TensorArrayStack/TensorArrayGatherV3 = TensorArrayGatherV3[_class=["loc:@style_encoder/bidirectional/TensorArray"], dtype=DT_FLOAT, element_shape=[?,128], _device="/job:localhost/replica:0/task:0/device:CPU:0"](style_encoder/bidirectional/TensorArray, style_encoder/bidirectional/TensorArrayStack/range, style_encoder/bidirectional/while/Exit_1)]]

Caused by op 'style_encoder/bidirectional/TensorArrayStack/TensorArrayGatherV3', defined at:
  File "/usr/local/lib/python3.6/threading.py", line 884, in _bootstrap
    self._bootstrap_inner()
  File "/usr/local/lib/python3.6/threading.py", line 916, in _bootstrap_inner
    self.run()
  File "/usr/local/lib/python3.6/site-packages/anyio/_backends/_asyncio.py", line 867, in run
    result = context.run(func, *args)
  File "/usr/local/lib/python3.6/site-packages/contextvars/__init__.py", line 38, in run
    return callable(*args, **kwargs)
  File "./main.py", line 616, in convert_markdown
    initialize_model()
  File "./main.py", line 78, in initialize_model
    style_embedding_size=256
  File "./train.py", line 56, in __init__
    super(StyleSynthesisModel, self).__init__(reader=None, **kwargs)
  File "./tf_base_model.py", line 110, in __init__
    self.graph = self.build_graph()
  File "./tf_base_model.py", line 401, in build_graph
    self.loss = self.calculate_loss()
  File "./train.py", line 81, in calculate_loss
    style_emb = style_encoder(self.ref_x)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/engine/topology.py", line 258, in __call__
    output = super(Layer, self).__call__(inputs, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/layers/base.py", line 696, in __call__
    outputs = self.call(inputs, *args, **kwargs)
  File "./train.py", line 26, in call
    x = self.bi_rnn(stroke_sequence)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/engine/topology.py", line 258, in __call__
    output = super(Layer, self).__call__(inputs, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/layers/base.py", line 696, in __call__
    outputs = self.call(inputs, *args, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/layers/wrappers.py", line 346, in call
    y = self.forward_layer.call(inputs, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/layers/recurrent.py", line 2056, in call
    inputs, mask=mask, training=training, initial_state=initial_state)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/layers/recurrent.py", line 613, in call
    input_length=timesteps)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/backend.py", line 2963, in rnn
    outputs = output_ta.stack()
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/ops/tensor_array_ops.py", line 893, in stack
    return self._implementation.stack(name=name)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/ops/tensor_array_ops.py", line 291, in stack
    return self.gather(math_ops.range(0, self.size()), name=name)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/ops/tensor_array_ops.py", line 305, in gather
    element_shape=element_shape)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/ops/gen_data_flow_ops.py", line 4186, in _tensor_array_gather_v3
    flow_in=flow_in, dtype=dtype, element_shape=element_shape, name=name)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/framework/op_def_library.py", line 787, in _apply_op_helper
    op_def=op_def)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/framework/ops.py", line 3271, in create_op
    op_def=op_def)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/framework/ops.py", line 1650, in __init__
    self._traceback = self._graph._extract_stack()  # pylint: disable=protected-access

UnimplementedError (see above for traceback): TensorArray has size zero, but element shape [?,128] is not fully defined. Currently only static shapes are supported when packing zero-size TensorArrays.
	 [[Node: style_encoder/bidirectional/TensorArrayStack/TensorArrayGatherV3 = TensorArrayGatherV3[_class=["loc:@style_encoder/bidirectional/TensorArray"], dtype=DT_FLOAT, element_shape=[?,128], _device="/job:localhost/replica:0/task:0/device:CPU:0"](style_encoder/bidirectional/TensorArray, style_encoder/bidirectional/TensorArrayStack/range, style_encoder/bidirectional/while/Exit_1)]]

[[05/11/2025 09:13:41 AM]] Processing failed for item 13: TensorArray has size zero, but element shape [?,128] is not fully defined. Currently only static shapes are supported when packing zero-size TensorArrays.
	 [[Node: style_encoder/bidirectional/TensorArrayStack/TensorArrayGatherV3 = TensorArrayGatherV3[_class=["loc:@style_encoder/bidirectional/TensorArray"], dtype=DT_FLOAT, element_shape=[?,128], _device="/job:localhost/replica:0/task:0/device:CPU:0"](style_encoder/bidirectional/TensorArray, style_encoder/bidirectional/TensorArrayStack/range, style_encoder/bidirectional/while/Exit_1)]]

Caused by op 'style_encoder/bidirectional/TensorArrayStack/TensorArrayGatherV3', defined at:
  File "/usr/local/lib/python3.6/threading.py", line 884, in _bootstrap
    self._bootstrap_inner()
  File "/usr/local/lib/python3.6/threading.py", line 916, in _bootstrap_inner
    self.run()
  File "/usr/local/lib/python3.6/site-packages/anyio/_backends/_asyncio.py", line 867, in run
    result = context.run(func, *args)
  File "/usr/local/lib/python3.6/site-packages/contextvars/__init__.py", line 38, in run
    return callable(*args, **kwargs)
  File "./main.py", line 616, in convert_markdown
    initialize_model()
  File "./main.py", line 78, in initialize_model
    style_embedding_size=256
  File "./train.py", line 56, in __init__
    super(StyleSynthesisModel, self).__init__(reader=None, **kwargs)
  File "./tf_base_model.py", line 110, in __init__
    self.graph = self.build_graph()
  File "./tf_base_model.py", line 401, in build_graph
    self.loss = self.calculate_loss()
  File "./train.py", line 81, in calculate_loss
    style_emb = style_encoder(self.ref_x)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/engine/topology.py", line 258, in __call__
    output = super(Layer, self).__call__(inputs, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/layers/base.py", line 696, in __call__
    outputs = self.call(inputs, *args, **kwargs)
  File "./train.py", line 26, in call
    x = self.bi_rnn(stroke_sequence)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/engine/topology.py", line 258, in __call__
    output = super(Layer, self).__call__(inputs, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/layers/base.py", line 696, in __call__
    outputs = self.call(inputs, *args, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/layers/wrappers.py", line 346, in call
    y = self.forward_layer.call(inputs, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/layers/recurrent.py", line 2056, in call
    inputs, mask=mask, training=training, initial_state=initial_state)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/layers/recurrent.py", line 613, in call
    input_length=timesteps)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/backend.py", line 2963, in rnn
    outputs = output_ta.stack()
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/ops/tensor_array_ops.py", line 893, in stack
    return self._implementation.stack(name=name)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/ops/tensor_array_ops.py", line 291, in stack
    return self.gather(math_ops.range(0, self.size()), name=name)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/ops/tensor_array_ops.py", line 305, in gather
    element_shape=element_shape)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/ops/gen_data_flow_ops.py", line 4186, in _tensor_array_gather_v3
    flow_in=flow_in, dtype=dtype, element_shape=element_shape, name=name)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/framework/op_def_library.py", line 787, in _apply_op_helper
    op_def=op_def)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/framework/ops.py", line 3271, in create_op
    op_def=op_def)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/framework/ops.py", line 1650, in __init__
    self._traceback = self._graph._extract_stack()  # pylint: disable=protected-access

UnimplementedError (see above for traceback): TensorArray has size zero, but element shape [?,128] is not fully defined. Currently only static shapes are supported when packing zero-size TensorArrays.
	 [[Node: style_encoder/bidirectional/TensorArrayStack/TensorArrayGatherV3 = TensorArrayGatherV3[_class=["loc:@style_encoder/bidirectional/TensorArray"], dtype=DT_FLOAT, element_shape=[?,128], _device="/job:localhost/replica:0/task:0/device:CPU:0"](style_encoder/bidirectional/TensorArray, style_encoder/bidirectional/TensorArrayStack/range, style_encoder/bidirectional/while/Exit_1)]]

[[05/11/2025 09:13:41 AM]] Error during model inference: TensorArray has size zero, but element shape [?,128] is not fully defined. Currently only static shapes are supported when packing zero-size TensorArrays.
	 [[Node: style_encoder/bidirectional/TensorArrayStack/TensorArrayGatherV3 = TensorArrayGatherV3[_class=["loc:@style_encoder/bidirectional/TensorArray"], dtype=DT_FLOAT, element_shape=[?,128], _device="/job:localhost/replica:0/task:0/device:CPU:0"](style_encoder/bidirectional/TensorArray, style_encoder/bidirectional/TensorArrayStack/range, style_encoder/bidirectional/while/Exit_1)]]

Caused by op 'style_encoder/bidirectional/TensorArrayStack/TensorArrayGatherV3', defined at:
  File "/usr/local/lib/python3.6/threading.py", line 884, in _bootstrap
    self._bootstrap_inner()
  File "/usr/local/lib/python3.6/threading.py", line 916, in _bootstrap_inner
    self.run()
  File "/usr/local/lib/python3.6/site-packages/anyio/_backends/_asyncio.py", line 867, in run
    result = context.run(func, *args)
  File "/usr/local/lib/python3.6/site-packages/contextvars/__init__.py", line 38, in run
    return callable(*args, **kwargs)
  File "./main.py", line 616, in convert_markdown
    initialize_model()
  File "./main.py", line 78, in initialize_model
    style_embedding_size=256
  File "./train.py", line 56, in __init__
    super(StyleSynthesisModel, self).__init__(reader=None, **kwargs)
  File "./tf_base_model.py", line 110, in __init__
    self.graph = self.build_graph()
  File "./tf_base_model.py", line 401, in build_graph
    self.loss = self.calculate_loss()
  File "./train.py", line 81, in calculate_loss
    style_emb = style_encoder(self.ref_x)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/engine/topology.py", line 258, in __call__
    output = super(Layer, self).__call__(inputs, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/layers/base.py", line 696, in __call__
    outputs = self.call(inputs, *args, **kwargs)
  File "./train.py", line 26, in call
    x = self.bi_rnn(stroke_sequence)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/engine/topology.py", line 258, in __call__
    output = super(Layer, self).__call__(inputs, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/layers/base.py", line 696, in __call__
    outputs = self.call(inputs, *args, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/layers/wrappers.py", line 346, in call
    y = self.forward_layer.call(inputs, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/layers/recurrent.py", line 2056, in call
    inputs, mask=mask, training=training, initial_state=initial_state)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/layers/recurrent.py", line 613, in call
    input_length=timesteps)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/backend.py", line 2963, in rnn
    outputs = output_ta.stack()
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/ops/tensor_array_ops.py", line 893, in stack
    return self._implementation.stack(name=name)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/ops/tensor_array_ops.py", line 291, in stack
    return self.gather(math_ops.range(0, self.size()), name=name)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/ops/tensor_array_ops.py", line 305, in gather
    element_shape=element_shape)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/ops/gen_data_flow_ops.py", line 4186, in _tensor_array_gather_v3
    flow_in=flow_in, dtype=dtype, element_shape=element_shape, name=name)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/framework/op_def_library.py", line 787, in _apply_op_helper
    op_def=op_def)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/framework/ops.py", line 3271, in create_op
    op_def=op_def)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/framework/ops.py", line 1650, in __init__
    self._traceback = self._graph._extract_stack()  # pylint: disable=protected-access

UnimplementedError (see above for traceback): TensorArray has size zero, but element shape [?,128] is not fully defined. Currently only static shapes are supported when packing zero-size TensorArrays.
	 [[Node: style_encoder/bidirectional/TensorArrayStack/TensorArrayGatherV3 = TensorArrayGatherV3[_class=["loc:@style_encoder/bidirectional/TensorArray"], dtype=DT_FLOAT, element_shape=[?,128], _device="/job:localhost/replica:0/task:0/device:CPU:0"](style_encoder/bidirectional/TensorArray, style_encoder/bidirectional/TensorArrayStack/range, style_encoder/bidirectional/while/Exit_1)]]

[[05/11/2025 09:13:41 AM]] Processing failed for item 14: TensorArray has size zero, but element shape [?,128] is not fully defined. Currently only static shapes are supported when packing zero-size TensorArrays.
	 [[Node: style_encoder/bidirectional/TensorArrayStack/TensorArrayGatherV3 = TensorArrayGatherV3[_class=["loc:@style_encoder/bidirectional/TensorArray"], dtype=DT_FLOAT, element_shape=[?,128], _device="/job:localhost/replica:0/task:0/device:CPU:0"](style_encoder/bidirectional/TensorArray, style_encoder/bidirectional/TensorArrayStack/range, style_encoder/bidirectional/while/Exit_1)]]

Caused by op 'style_encoder/bidirectional/TensorArrayStack/TensorArrayGatherV3', defined at:
  File "/usr/local/lib/python3.6/threading.py", line 884, in _bootstrap
    self._bootstrap_inner()
  File "/usr/local/lib/python3.6/threading.py", line 916, in _bootstrap_inner
    self.run()
  File "/usr/local/lib/python3.6/site-packages/anyio/_backends/_asyncio.py", line 867, in run
    result = context.run(func, *args)
  File "/usr/local/lib/python3.6/site-packages/contextvars/__init__.py", line 38, in run
    return callable(*args, **kwargs)
  File "./main.py", line 616, in convert_markdown
    initialize_model()
  File "./main.py", line 78, in initialize_model
    style_embedding_size=256
  File "./train.py", line 56, in __init__
    super(StyleSynthesisModel, self).__init__(reader=None, **kwargs)
  File "./tf_base_model.py", line 110, in __init__
    self.graph = self.build_graph()
  File "./tf_base_model.py", line 401, in build_graph
    self.loss = self.calculate_loss()
  File "./train.py", line 81, in calculate_loss
    style_emb = style_encoder(self.ref_x)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/engine/topology.py", line 258, in __call__
    output = super(Layer, self).__call__(inputs, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/layers/base.py", line 696, in __call__
    outputs = self.call(inputs, *args, **kwargs)
  File "./train.py", line 26, in call
    x = self.bi_rnn(stroke_sequence)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/engine/topology.py", line 258, in __call__
    output = super(Layer, self).__call__(inputs, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/layers/base.py", line 696, in __call__
    outputs = self.call(inputs, *args, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/layers/wrappers.py", line 346, in call
    y = self.forward_layer.call(inputs, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/layers/recurrent.py", line 2056, in call
    inputs, mask=mask, training=training, initial_state=initial_state)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/layers/recurrent.py", line 613, in call
    input_length=timesteps)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/keras/_impl/keras/backend.py", line 2963, in rnn
    outputs = output_ta.stack()
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/ops/tensor_array_ops.py", line 893, in stack
    return self._implementation.stack(name=name)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/ops/tensor_array_ops.py", line 291, in stack
    return self.gather(math_ops.range(0, self.size()), name=name)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/ops/tensor_array_ops.py", line 305, in gather
    element_shape=element_shape)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/ops/gen_data_flow_ops.py", line 4186, in _tensor_array_gather_v3
    flow_in=flow_in, dtype=dtype, element_shape=element_shape, name=name)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/framework/op_def_library.py", line 787, in _apply_op_helper
    op_def=op_def)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/framework/ops.py", line 3271, in create_op
    op_def=op_def)
  File "/usr/local/lib/python3.6/site-packages/tensorflow/python/framework/ops.py", line 1650, in __init__
    self._traceback = self._graph._extract_stack()  # pylint: disable=protected-access

UnimplementedError (see above for traceback): TensorArray has size zero, but element shape [?,128] is not fully defined. Currently only static shapes are supported when packing zero-size TensorArrays.
	 [[Node: style_encoder/bidirectional/TensorArrayStack/TensorArrayGatherV3 = TensorArrayGatherV3[_class=["loc:@style_encoder/bidirectional/TensorArray"], dtype=DT_FLOAT, element_shape=[?,128], _device="/job:localhost/replica:0/task:0/device:CPU:0"](style_encoder/bidirectional/TensorArray, style_encoder/bidirectional/TensorArrayStack/range, style_encoder/bidirectional/while/Exit_1)]]

[[05/11/2025 09:16:15 AM]] Shutting down
[[05/11/2025 09:16:15 AM]] Waiting for application shutdown.
[[05/11/2025 09:16:15 AM]] Application shutdown complete.
[[05/11/2025 09:16:15 AM]] Finished server process [1527]
